pkg/ccl/testccl/workload/schemachange/schemachange_test: TestWorkload failed [column is referenced by the primary key]

pkg/ccl/testccl/workload/schemachange/schemachange_test.TestWorkload [failed](https://tanzanite.cluster.engflow.com/invocations/default/6524eb80-3e30-4caf-8695-0ca069d54750?testReportRun=15&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL3Rlc3RjY2wvd29ya2xvYWQvc2NoZW1hY2hhbmdlOnNjaGVtYWNoYW5nZV90ZXN0) on master @ [fd4b1464dbd6e385c6e51af26fe294fd2023a259](https://github.com/cockroachdb/cockroach/commits/fd4b1464dbd6e385c6e51af26fe294fd2023a259):


```
 ],
 "expectedExecErrors": "",
 "expectedCommitErrors": "",
 "message": ""
}
{
 "workerId": 1,
 "clientTimestamp": "10:42:15.265757",
 "ops": [
  "BEGIN",
  {
   "sql": "INSERT INTO public.table_w0_114 (col114_w0_115, col114_w0_116, col114_w0_117, col114_w0_118, col114_w0_119, col114_w0_120, col114_w0_121, col114_w0_122, col114_w0_123) VALUES ('2008-10-09 17:44:15.00034+00':::TIMESTAMPTZ,true,'[9.0358667e-13,-3.4309166e+09,-5.702542e+18,-1.6061066e+15,-1.5112797e+13,0.0005485193,2.260893e-36,-2.8334441e-33,4.1310312e+21,-3.3332536e-15,7.4291936e-16,1.4037446e-14,-8.855e+08,-1.5578073e-13,-1.6864728e+07,1.6301872e-37,5054.896,3.489014e+20,-5.7241514e-32,-2.0941657e+15,-7.123907e-15,-4.3096166e-35,-9.991583e-10,3.8059188e-25,4.8464926e-35,2.6411554e+14,-1.3334522e-10,2.7370267e-17,4.7165612e-33,-5.6856124e-24,-410.49393,-4.600507e-17,-2.7737055e-30,-1.34014925e+20,1.1237477e+21,-2.1920374e+13,1.5820842e-31,-1.02322065e+12,-3.086148e+07,-0.063915834,2.0506294e+08,-4.8242285e+06,7.58168e+28,1.7286934e-28,2.2108162e+21,-1.5400126e+10,3.832311e-06,8.50227e+27,1.7622765e-19,1.0430245e-21,12.35219,-1.658993e+19,1.0874329e+38,-2.879631e+35,-1.09073124e-29,-3.6662918e+36,-1.0994739e+38,1.9559275e-35,-4.478563e+25,-1.68296e-23,-1.4943551e+10,1.2364369e-20,9.136029e-36,-5.4494053e-21,-7.151454e+18,-2.906809e-05,3.7741215e+06,8.597978e+09,1.7090786e+20,1.5389726e-11,7.541392e-35,-129913.29,7.327881e+23,5.037216e+13,-1.9008174e-38,8.9680115e+26,-1.1207722e+17,-7.88013e-19,1.0849559e+18,-4.9861476e-11,-5.4781187e-34,-2.6578093e+18,3.6527996e+15,9.8513495e+28,-1.0019197,-6.017645e-25,3.5998376e-08,2.003772e+07,2.4445972e-09,-1.1914581e+11,-2.7046404e-13,7.6481313e+28,-3.2531195e-16,9.600171e+18,-1.475867e-18,-2.3644859e-29,-1.5313104e-30,-2.3269975e+25,1.4840077e-30,8.722157e-34,-1.7533083e-12,1.1766418e+37,-2.566424e+10,-63.558968,-5.015285e+25,-3.3266097e-09,1.4328473e-25,-8.890576e-29,-6.3425956e-34,-2.5368166e+23,-2.3701291e-14,8.911913e+32,-775.23755,-9.913961e-16,0.16398002,-2.8086398e+23,0.5725136,-5.893592e-10,2.2007854e-19,2.140468e+35,-4.6585696e+15,3.5765395e-24,4.058775e-32,-1.03806826e+30,5.4841242e+29,6.4415605e-11,1.2071555e+37,-5.231801e-28,-5.2173967e-09,1.1547684e-13,1.2507825e-08,1.7004821e-10,1.6777969e-12,-5.428493e-22,-9.785736e+06,1.2684393e-13,-2.49456e+32,1.2859907e+33,2.3331516e+23,-2.102768e+25,5.539293e-34,-795004.3,1.8807643e+34,201368.77,-7.568003e-32,-7.4499326e-06,-2.8562159e-15,-211576.4,1.7034394e-31,9.187227e-34,-2.805247e-25,7.7387154e+36,-2.4937362e-38,-1.75872e-13,-3.304381e-22,-2.637234e+35,-8.3340156e+18,1.7705156e-31,1.2279136e+17,1.4890009e-33,-1.520818e+35,-8.222206e+16,-0.08444948,1.7731416e-15,1.4767651e-06,2.5028737e-17,5.2631827e+35,-1.7716661e-31,-2.743967e-18,7.380589e+07,2.8334716e-06,-1.6777626e+06,-3.7904297e-34,5.9500976e-37,-1.0387911e-36,-1.19800705e-20,2.1610567e+18,-8.470109e+09,-4.822818e+15,1.3818039e+26,-3.4005423e+18,-5.9862436e-15,-2.2374956e-15,7.597262e+21,-1.2438611e+37,-4.4504033e-20,1.7488135e-18,-4.6021458e-05,-6.991906e+06,382.5175,-1.1412078e-18,4.751622e+30,3.0429639e+18,-3.3011136e+31,-1.4213732e-11,2.2695508e-10,2.5670074e-31,-7.690126e+32,7.643578e+10,-5.114262e-20,-2.9230593e+37,9.555987e-11,-41097.504,3.7717705e+15,7.102152e+06,-6.7908714e-32,-3.684789e+15,1.3422583e+26,2.308024e-07,2.2862383e-38,2.56238e+16,-1.1199515e-16,2.413519e-16,-4.991595e-23,-4.9437844e+16,1.6148282e+14,-9.906874e-14,2.2829602e+23,-8.950101e+10,-3.8668985e-19,-1.0937327e+16,-2.6079075e-33,-4.217834e-27,-7.2648395e+06,-1.7170522e-32,1.2648606e+25,-4.889932,-6.820478e+37,-1.611592e-34,2.6365323e-36,-5.0740706e+35,1.9849133e+32,1704.8429,2.5660459e+11,-3.030718e-06,-7.1524677,9.952962e+20,0.017957881,2.638158e-18,1.717934e-16,7.6844496e+25,-1.5456342e-38,-529866.8,-3.0936617e+23,-1.1161362e+08,-7.392251e-13,3.154131e+11,-5.1977227e-17,-1.5235975e+06,2.2632506e+30,-4.093219e+15,2.1849518e+25,3.8303978e-20,-8.090943e+27,-9.590534e-07,-5.9580278e-24,-5.030006e+14,9.734779e-29,-3.215119e-17,-3.588458e-14,2.0694156e-10,-2.6796744e-21,-4.0665214e+12,-7.034328e-27,-1.8557752e+33,-13027.438,2.1375705e-17,-3.7160377e+29,25953.537,3.9132924e+12,3027.1138,-0.09200257,3.1337888e+06,-8.206383e+28,5.2665406e+19,2.9036228e+06,8.5340645e+34,1.6051696e+06,2.256563e+20,5.825165e-30,3.127125e+25,8.854812e+22,0.000103831946,4.679048e-35,1.443474e-37,-4.456843e+23,-1.1396195e+32,-4.4236213e+35,-24.863817,1.8323472e+34,9.8731955e+26,4.0383818e-21,-1.0987445e+38,-0.00047068362,0.2864095,4.656568e+21,-1.7416782,-5.175339e-30,2.0646906e+30,2.979286e+33,7.5662107e+18,-4992.0615,8.657748e-38,-0.000148764,0.029813932,-4.7071425e+22,5.253037e-18,9.9594945e-23,-9.290221e-09,-1.7945797e+10,8.719049e-19,5.2382247e-13,-2.829769e-33,1.6240188e-30,2.0844354e-19,6.128144e+28,3.8912244e+12,3.292972e-07,1.7253452e+36,-4.601939e-19,5.9878434e+23,-2.1236855e-37,1.443753e+27,1.1996221e-30,6.4607885e-27,-1.2230731e-20,9.830327e-34,-2.2142507e+11,0.12791139,-6.7117677e+24,7.2937603e-29,-1.7744674e+31,-3.363203e-07,-1.9047674e+15,-1.2968186e+38,-0.03994344,3.7473185e-22,6.1166607e+29,5.1417977e+20,2.3510887e+15,0.0075209746,-5.1560974e-24,-32.777184,-4.7887336e+35,-8.837395e-36,6.2970037e+35,3.9350275e-31,-0.02339917,2193.178,2.7054692e-27,3.386353e+17,7.210374e+37,4.418707e+37,-2.2634725e+38,5.15376e-09,1.1710028e+14,-2199.79,1.6117899e+24,-9.622651e+11,2.206092e+13,3.5376427e+10,-4.3444212e+35,3.7230307e+13,-7.6571093e+12,-3.8060308e+32,1.7524551e+37,-0.01496414,-1.2481028e-25,3.5215556e-31,6.367752e-11,-1.6654834e+33,-3.9328248e+12,-338.07578,4.3139122e-33,-3.6575098e+19,3.3588069e-21,1.6614564e-19,-0.055373985,3.1732033e+12,9.245109e-39,1.0025511e+28,-6.212794e+06,4.8776616e+07,-2.5710651e-38,1.0620241e-20,-2.0160661e-17,-9.414326e+21,2.3239763e-17,-7.656642e-25,-4.68217e+30,0.18021628,2.0776608e+24,-1.17689615e+27,3.827461e-08,6.279983e+17,-0.008445552,-7.3181504e-18,0.98714674,81.532005,1.5198156e+06,2.6703087e+13,-118.01566,3.4658174e+26,-9.940482e-37,9.7902045e+23,4.9537563,1.5952565e-12,-18239.615,101.58234,-0.058863297,6.7857296e+13,1.6785099e+18,4.2312245,130522.695,0.00033741762,-5.652359e-25,-7.179047e+30,-1.1134557e+38,-5.249404e-16,-9.637216e-28,-4.4953035e-27,-3.1674938e-35,2.2926658e+23,-4.6791513e-37,7.881582e-30,2.423835e+38,8.971695e-24,9.981755e+37,8.097569e-21,2.1991594e-22,2.8011482e-07,7.879772e-25,-1.0238663e+30,-8.846829e-33,6.6328292e+16,1197.3298,11719.01,4.5328715e+29,-4.5724397e-17,-6.300603e-17,-1.6526481e+24,7.012477e+25,2.0066774e+33,-13.910499,-7.084372e+32,1.5360947e-12,8.163641e+17,1.7861299e-23,4.1174683e+21,-3.0527324e-18,-9.252878e+28,-2.0404503e+13,-2.8954322e-21,-1.1481472e-16,1.5029111e+26,-0.11774601,5.426127,4.5783e+21,-2.4701628e+33,1.0060433e-27,-121023.88,-3.6821076e+21,-2.8861303e+31,0.0013330987,-3.288755e-10,9.494422e+19,-2.4959128e-28,-3.846992e+35,7.538531e-29,4.1272386e+21,2.0016686e+18,-3.1193395e-11,-4.992963e+27,-1.6834829e-13,6.013368e-24,-7.1346106e-12,-1.9322706e+17,-1.8598962,7.392056e-24,1.2037879e+32,-5.7068293e+28,-16391.049,-1.17017015e+21,-1.0840635e+27,-7.703184e+23,-235.29573,1.0132556e-28,-0.0016061293,-2.8462375e-18,-0.0018799669,2.1131253e-11,-5.5181134e-11,0.35932052,1.0923471e+34,-5.669017e+10,0.9966688,4.497856e+28,4.046662e-30,5.0669312e+30,9.443785e+36,4.312524e+25,-3.0382127e-36,-280751.72,-7.9770685e-22,1.0156424e-10,3.0374445e-31,1.0436715e-30,4.7706006e-08,4.696587e-10,-1.9026818e-20,-3.948181e+17,1.9005785,3.1037865e+19,-1.2468586e+07,1.07269804e-13,-1.1987724e+26,-4.5511587e+17,-2.3927003e-29,-6.33963e+12,6.356616e-28,-2.6239826e+21,-1.1356187e+15,-3.7515277e-20,9.2735095e+10,-5564.268,-2.2098886e+12,-1.0187484e+32,-9.20629e+19,4.4531834e-32,1.5162387e-07,10903.531,8.069676e-26,-3.2071085e-29,-4.572062e+12,-4.539086e+32,-1.2190939e+23,-0.00012270304,-1.7995494e+37,1.1544211e-23,2.5372014e-20,5.455224e-16,-1.2399951e-16,212589.81,-1.1419069e+17,1.3255868e+17,-4.3545007e+16,2.7505362e+28,-5.6335437e-25,4.388496e-08,-8.9589724e+11,-9.800109e-37,-1.0773021e-25,-1.4455396e-09,1.5171666e+24,-3.3879997e-16,106723.195,-1.1744827e+17,1.2255632e-31,-7.5208706e+37,4.3508135e+16,9.714305e-09,-6.067258e-07,-0.12394691,2.2321782e+38,-1.092181e+37,1.305354e-22,-8.367095e-28,-1.3599575e-27,8.749671e+18,-5.8782456e+35,-1.1302231e+12,4.0749954e-19,-1.869878e-36,-9.4103435e+23,2.5684643e-05,-1.2050185e-27,2.97491e+08,-3.8508777e-08,8.690798e+27,3.264695e+25,9.261995e-24,-28493.693,-8.846566e+15,6.7098722e+10,-9.134898e+32,-6.005573e+37,8.080462e+27,1.5388456e-18,-7.592721e+19,1.13625405e+20,-6.873271e+23,-5.490177e-21,4.7462497e+32,1.7424989e+21,8.2471615e+26,-3.0327987e+11,-3.481123e-11,1.9740453,1.5462881e-18,-4.8985023e+23,1.3324504e+19,0.011185048,-86.23392,-2.6611487e-27,-4.165927e-09,-2.7323505e+12,-3.6386518e+33,1.3793679e-19,-2.939402e+35,-3054.9375,1.0820992e-38,9.4345515e+20,-12.271438,-4.22491e-29,-1.005484e-16,2.5315918e+10,1.731663e+37,3.3793534e+11,1.1381893e-20,-2.1163598e+10,-9.4986956e+26,3.543344e-36,4.4696487e-36,6.4707774e-11,1.2105572e-25,4.2420324e-05,-4.4841693e-11,4.9470545e+31,-6.4364987e-32,-7.384242e+17,-1.9533596e+28,2.1107083e-26,-2.6628807e-20,-1.7153465e+22,-1.4851247e-06,6.0603056e+14,5.20627e+11,-398415.38,2.0664285e+25,-1.829732e+15,-6.911574e+26,463.5852,-2.6273302e-20,3.6790777e-08,1.177333e-22,118.74396,-8.791852e+27,1.081596e+14,6.7711304e-13,8.668046e+31,1.7243436e-21,-1.8118201e-10,-1.4551113e-13,-7.2642315e+06,1.2140671e+36,-1.8853514e-11,-9.930395e-20,-1.5182248e+21,7.3869296e-14,-4.7440315e-38,-1.1028267e-31,3.7278056e-30,-3.7152624e-35,-1.7992546e+30,-3.2515709e+10,0.0064067496,4.0281614e+31,-4.5910195e+27,-1.2465454e-29,-1.9873747e-17,3.912352e+33,3.124938e+17,-2.251106e-32,-3.6618197e+11,-7.894719e+15,-2.990133e+24,-6.13836e-12,2.5217876e-34,2.33358e+24,-1.5930427e-19,4.94202e+26,-6.2295635e+18,0.0062832674,-2.2234284e-23,-1.8845114e-05,-5.748838e+34,5.3016525e-11,3.6053883e-16,3.1524432e+06,-2.1033131e+14,1.8198389e+22,0.19759281,-4.0447513e+32,9.773167e-32,-7.5840294e+20,-1.8164937e-09,8.191159e-30,-2.3038119e-15,3.3278808e+24,8.7523905e-31,1.1787825e-15,-64431.934,-4.7735695e-09,-3.382737e+24,-9.546245e-08,7.0254253e+31,3.2085734e-31,2.801297e+16,-1.6768918e+14,-4.4734924e+34,-1.6345684e-32,2.4197387e+08,2.0062405e+18,7.723102e-11,2.8157085e-21,-2.8104225e+17,1.4189742e+26,6.633552e-31,0.00066625717,-1.1898878e+18,2.3848884e+36,4.810281e-34,-1.33281155e+20,-4.2502077e-30,9.122994e-26,-1.3884973e-23,0.0019905074,-4.5406067e+11,3.4625774e-26,4.160494e-19,1970.3739,-3.3941552e+33,8.8616874e-30,-2.1516828e-29,1.8436492e-35,-6.8825836e+15,3.2090356e+13,4.6888633e-34,1.2555683e+20,-2.6256667e+35,3.6706047e+26,2.2249245e-15,8.361787e-16,1.473086e-06,1.937448e-06,-7.6037417e+09,3.450385e+12,-5.4261652e-17,-5.326394e+07,1.1359197e-18,4.5367326e-32,5.868136e+26,69543.42,-3.0160754e-31,-5.0530585e-34,-2.7472549e+20,-2.854313e-18,3.0476177e-21,2.7891474e-25,-7.1558037e-13,-0.0015534803,4.557232e-15,-6.391435e-33,-4.7545787e+36,-2.841347e-20,1.1540187e-34,4.242688e+21,-3.325513e+25,-1.1425642e-22,2.0425433e-26,-983549.7,-1.7350667e-19,8.641788e+08,1.785137e+10,-7.943621e-20,-1.6773502e-08,-5.948613e-13,6.533517e-31,-1.4895368e-28,1.5315325e+34,75742.59,1.1643687,0.14466454,1.0511115e+07,-1.0803308e+11,-3.653196e-28,4.7511325,-4.202317e-31,4.3946876e+31,1.8858245e-07,-1.7145335e-22,-1.1627035e-17,-2.2174799e+21,-1.6887097e+31,3.761744e-05,9.86916e-09,-4.5534894e+24,-1.8951681e-30,0.0017147999,6.2941628,0.011842571]':::VECTOR,ARRAY['47 years 9 mons 871 days 21:35:45.384235':::INTERVAL,'38 years 2 mons 23 days 05:01:41.398135':::INTERVAL,'37 years 3 mons 81 days 11:15:30.58206':::INTERVAL],20:::OID,B'0000000000000000000000000000000000000000000000000','N'::BPCHAR,'\\x757fe24a0aa56004':::BYTES,'{\"TFJD\": \"bar\", \"bar\": false, \"c\": null, \"dfE\": 1.9080125800644232}':::JSONB),('2022-09-30 03:30:18.000606+00':::TIMESTAMPTZ,true,'[6.409425e+25,3.6434082e+31,6.452742e-30,4.278626e+20,9.1546165e+29,5.232394e-13,-5.9892622e+32,-1.7437956e+18,1.9945048e+14,0.00020801357,1.7518786e-27,-1.3111971e+07,-3.168201e-11,1.0325253e+37,6.847825e+29,4.3741256e+07,1.8900394e-26,-3.3413987e+30,-2.0876458e+27,-3.354271e+14,1.756835e+07,5.9454926e+26,5.907174e+19,-8.7793353e-32,-3.4145627e+37,-1.176612e-13,-2.1340416e-05,1.3078191e-28,-2.2745973e-29,-2.5695834e-11,-1.5603702e-31,-3.0553905e+31,-9.239689e-21,1.19906465e-26,-3.8877308,3.1968012e+18,-5.4084093e-17,3.2379218e+10,2.679055e-30,-8880.141,-4.1199876e-14,-4.3538764e-24,1.3894019e+10,1.2734267e-28,2.4978894e+08,-2.4093644e+24,-0.0035474715,-1.0915707e-07,-2.5647498e-38,1.1738925e-08,1.7812345e+12,-2.394298e+06,-4.0742652e+33,-4.1109588e-25,-4.416816e+22,-1.568782e-35,1.721442e-26,7.6588406e+30,-2.5749802e-13,-7.934664e-13,1.7605146e-26,-2.812705e+15,-1.0760058e-06,-28699.588,2.9254027e+26,-3.7635087e-22,-1.9807343e+21,8.610366e-09,-2.2650792e-37,1.5622228e+32,5.6691153e+17,-1.5532043e-09,-2.5997626e+30,-2.9503773e-24,2.2749547e-24,1.2148537e-06,-1.7802642e+34,4.4459826e+19,6.847213e+36,9.954279e-30,0.45693383,2.5591728e-05,-5.928431e-27,20905.57,6.209129e-35,4.098308e-05,-2.0360334e-35,4.425922e-36,5.39827e-17,7.170871e-17,-1.8271333e-17,6.632995e-08,-6.881599e+32,-1.1115994e-14,-0.018554829,4.9903232e+33,4.8143387e+37,-1.9926875e+27,1.5968617e+34,-4.4755686e+33,-3.5265382e-06,-8.1586506e+18,4.2663325e+08,-2.9401302e+07,-7.510406e+34,2.1581886e+25,-1.3450746e-38,-5.0424043e+35,4.060141e-30,-1.0618895e+30,2.3959925e+18,1.5279352e+32,1.6251491e+28,13848.991,399742.7,8.296528e+35,549939.6,-1.4778866e+33,-2.856206e+33,5.537973e+32,5.7782116e+12,5.6320216e+21,-4.3343253e-30,1.2249219e+08,-1.7154928e+11,-1.2126579e+36,-2.8549378e-33,9.183152,-2.1973324e-25,1.1749873e-07,3.2238572e-15,0.03648405,3.040162e-14,2.5120104e+37,-2.014376e-36,1.0400487e+33,4.2850013e+37,3.5512507e-33,-2.9953663e+26,-5.669494e-25,-7.953449e+06,-97759.195,-7.002232e-21,-3.6960176e+08,-1.0612679e+31,3.8352957e-32,-3.519311e+29,9.280786e-19,4.4050607e-06,-3.9345328e-26,-4.0969824e-13,-2.0105804e+15,169.12262,-0.3029891,5.4725935e-27,-4.199639e-29,-1.649799e-28,-0.40945706,5.496309e-11,-2.2278716e+20,-1.216542e-35,7.543349e+27,1776.9802,1.8851252e-06,-0.0018083686,2.0911694e-25,3.7815804e+31,-6.0699454e-19,0.00016423309,-1.8730844e+28,1.7605673e-23,-8.781139e+16,-3.705156e+25,1.5509387e-10,4.2560317e-27,-4.74427e+24,2.0850284e+33,-9.312264e+36,-9.117179e+37,-3.8170067e+08,-2.8776868e+14,6.752491e-13,-2.7135719e-11,2.3556612e+26,-11777.734,-6.577574e+23,8.101512e+30,-5.1270903e+20,613423.94,7.790652e-20,9.124838e-07,7.3303106e+10,-6.851088e-20,-2886.3699,-1.417048e+18,-1.4223033,-1.6334054e+09,-6902.098,-0.82066643,4.7915985e-17,3.74344e+22,1.6413915e-28,1.7417855e-35,-391539.72,1.9781834e-29,6.5825922e-24,-3.9608707e+27,-1.897958e+24,-7.467104e+07,-6.5367376e-37,2589.9333,5.49261e-20]':::VECTOR,ARRAY['11 years 3 mons 757 days 12:22:09.569999':::INTERVAL,'63 years 3 mons 697 days 00:27:55.49021':::INTERVAL,'-41 years -1 mons -271 days -03:52:36.991338':::INTERVAL,'42 years 8 mons 572 days 09:16:39.82339':::INTERVAL],1560:::OID,B'0000111101111001101111011011001100001001010111000','/'::BPCHAR,'\\x9879':::BYTES,'[{\"c\": null}, false]':::JSONB)",
   "potentialExecErr": "23502,23505,23514"
  },
  "COMMIT"
 ],
 "expectedExecErrors": "",
 "expectedCommitErrors": "",
 "message": "TXN RETRY ERROR; ERROR: restart transaction: TransactionRetryWithProtoRefreshError: TransactionRetryError: retry txn (RETRY_SERIALIZABLE - failed preemptive refresh due to encountered recently written committed value /Table/3/1/125/2/1 @1728556935.548962430,0): \"sql txn\" meta={id=bb08f8c9 key=/Table/125/1/B0000000000000000000000000000000000000000000000000/0 iso=Serializable pri=0.03689057 epo=0 ts=1728556936.314614155,1 min=1728556935.262633868,0 seq=2199} lock=true stat=PENDING rts=1728556935.262633868,0 wto=false gul=1728556935.762633868,0 (SQLSTATE 40001)"
}
{
 "workerId": 0,
 "clientTimestamp": "10:42:17.274391",
 "ops": [
  "BEGIN",
  {
   "sql": "INSERT INTO public.table_w0_199 (col199_w0_200, col199_w0_201, col199_w0_202, col199_w0_203, col199_w0_204, col199_w0_205, col199_w0_206, col199_w0_207, col199_w0_208) VALUES (ARRAY['1989-07-16':::DATE,'1990-11-08':::DATE],NULL,72:::INT8,'{\"2eJZqtz\": [true, false], \"DjR6\": [{\"bar\": \"oFdGL0AEs\", \"fRjiL3\": null}, 0.04636229031106226], \"c\": null, \"foo\": true, \"nCv9\": false}':::JSONB,'{\"NrMQAtkc1\": [false], \"ZN\": 0.6653806841729224, \"foo\": 0.6864803322555988}':::JSONB,''::STRING,40:::INT8,'BOX(-1.0211211394815454 -0.2108782769530062,0.3653613029529523 1.0448209708178364)':::BOX2D,3220:::OID),(ARRAY['1994-02-01':::DATE,'1996-05-08':::DATE,'1982-09-04':::DATE],e'\\x10,\u003c88$H'::NAME:::NAME,83:::INT8,'[true]':::JSONB,'[[null], false, false]':::JSONB,'\"'::STRING,(-19):::INT8,'BOX(-0.6262332907773915 -0.5621078247977136,-0.5095408924854397 0.05769354719712236)':::BOX2D,4096:::OID),(ARRAY[]:::DATE[],e'\\x04\\x17oU'::NAME:::NAME,(-37):::INT8,'{\"QXaRD\": false, \"a\": {\"b\": null}, \"b\": false, \"foo\": [true]}':::JSONB,'[[[{\"PZS2IxkS\": null, \"a\": false}, null], false, \"zh5zeVb\"], true]':::JSONB,e'\\blFI'::STRING,(-42):::INT8,'BOX(-10 -10,10 10)':::BOX2D,90006:::OID)",
   "potentialExecErr": "23502,23505,23514"
  },
  "COMMIT"
 ],
 "expectedExecErrors": "",
 "expectedCommitErrors": "",
 "message": ""
}
{
 "workerId": 0,
 "clientTimestamp": "10:42:22.220515",
 "ops": [
  "BEGIN",
  {
   "sql": "INSERT INTO public.table_w0_141 (col114_w0_115, col114_w0_116, col_w0_114_w0_140, col114_w0_118, col114_w0_119, col114_w0_120, col114_w0_121, col114_w0_122, col114_w0_123) VALUES ('1984-07-18 22:00:14.00062+00':::TIMESTAMPTZ,true,'[2.109995e-19,-2.0164917e+37,2.1742848e+10,-2.2743156e-07,-3.9706328e-35,3.055699e+07,7.488622e-20,-4.2030976e-08,-2.7309688e+11,7.248068e-17,-1.5384919e+26,7.111744e-30,5.0246886e-12,-4.0024845e+34,2.899199e+31,-1.4986174e-05,1.8943767e-36,1.1737177e+33,-3.4268172e-12,-1.037283e-37,9.6348634e+08,1.1438229e+21,1.4862355e-33,-1.020657e+17,0.008289162,2.415107e-07,-3.8474356e+33,2.2558517e+14,-40.618572,-7.3758907,-7.609519e+14,3.3475843e-07,-1.9682731e-31,2.547579e+20,-1.2505185e-31,-9.25842e-08,1.04733046e+21,4.1676392e+24,1.9975804e+23,-3.8838315e+32,4.993398e+15,-3.1907203e-14,6.518116e-20,2.5298673e-06,0.30162796,1.5189423e-19,858.83154,7.698795e+09,-2.1314066e-18,-565.26416,-4.6417898e+19,2.7861582e-16,-1.0797916e-30,-2.6703558e+22,8.975371e-19,-3.0201133e-24,4.772398e-31,1.6908061e+16,6.907177e+17,3.639301e+27,6.6314636e+26,4.8644515e-16,-7.1387588e-12,3.7941053e-38,-5.6170516e-06,-2259.6313,-1.1677908e-22,-63.006527,2.57023e-38,-4.873924e-23,2.780341e+33,-8.3533055e+06,-3.8580134e+08,-3.6942681e-06,2.7005087e-24,7.3766535e-07,-4.221519e+17,3.8267254e-22,1.9684533e-38,1.9466146e-08,-7.029214e+28,-4.6727587e+13,22.483105,-6.4253175e+06,1.6407681e+19,1.1335113e-09,-1.1877455e+22,-2.495349e-34,4.437819e-14,1.772026e+12,-7.914192e-16,-1.5748186e-07,2.886891e-23,2.471524e+15,-0.01619947,1.5890814e+24,1.186921e-22,2.4911659e-08,-230.09473,-0.0014834631,-1.4963267e+37,-3.629418e-21,-9.029277e+19,1.6166463e-23,-1.03119446e+34,-6.338808e-38,-7.140047e-25,-0.8340269,3.46511e-06,-0.0015976293,1.3307429e+31,-3.2284178e-17,1.3337275e-19,-6.782283e+10,17230.838,-5.7618774e-37,-6.980979e-05,-3.1214717e-18,-2.55212e-34,1.7857628e+30,-1.1323365e-27,1.5816687e+22,2882.997,4.49972e-07,-1.8608099e-32,-280.29434,1.811856e+20,-503126.4,3.8319647e-22,-3.1196912e+33,-0.0011398563,1.2148659e-32,-3.3539158e-07,5.753761e+23,-2.3992525e-06,-0.027722979,9.567665e-24,6.7203735e+13,-1.5181453e-18,2.006423e-37,-4.060146e-21,2.1558954e+14,1.1080167e-11,1.2040312e+31,7.5776e+06,-1.8673053e+25,9.599058e-19,7.152886e+15,-2.9591789e+26,-1.7698559e+06,1.1752841e+37,1.2162363e+20,-1.2610534e-36,4621.595,9.398385e-32,0.0003638646,1.4739982e+27,1.3017965e-23,-2.951411e-06,-334.19946,4.5719093e-33,3.7476515e+12,3.454707,-3.3621335e-26,1.0781595e+38,3.915969e-21,-1.8653783e-14,-1.0687839e-14,1.1369352e+11,-1.6132896e-09,1.934586e+28,-1.6430592e-34,9.911801e-29,-8.7535315e+35,-3.0336155e-27,-3.1747747e-25,4.8741402e-23,-6.4178485e-24,1.255653e-06,-8.401446e-39,-3.6340748e-26,2.1040208e-11,-3.8195126,-3.890855e+33,-1.5288931e-37,-1.706477e+11,-11.410254,3.2057097e+38,-1.27964e-31,-1.4209548e-32,0.028801685,4.3294758e+29,5.2479985e-27,1.712408e+27,-467808.4,3.2504173e-09,1.7475314e+11,1.28404835e-17,7.823684e+27,9.8419045e-30,1.5785784e-19,1.4853523e-22,8.6279116e+33,1.0734287e-27,5.2889377e-26,2.0156097e+26,-116945.57,-3.1915557e-19,-5.234125e+27,3.4496715e-15,1.0983209e+09,-1.3949537e-07,-1.9296311e+38,-18453.342,5.0276022e-26,3.3529842e-11,-1.1023483e+16,2.6291002e-22,-1.3866816e-05,1.8643176e-22,8.219512e+27,3.155042e+06,2.4447787e-20,-1.9209553e-38,2.336754e-30,5.378626e-26,6.988177e+25,-1.2856026e+26,3.8796306e-16,2.872346e+11,-7.873408e+06,1.7547962e-17,7.4616774e+32,-1.3640808e-28,-7.2303665e-13,-7.991432e+22,-2.7409462e-06,-2.2715525e+36,3.0855491e+29,-2.6208873e+36,1.1074285e+21,-8.112895e+09,8.531118e-27,8.3465455e-17,-1.489745e+22,-3.195415e+23,-3.5430023e+11,-1.5066663e+23,2.2334126e+29,4.0310654e-19,13.756806,5.5414346e-36,9.461684e-22,9.234196e-05,-1.2105669e+11,1.5194291e-07,-3.9798746e+23,-2.225712e+29,2.7279663e-08,-2.0457336e-28,-104.15809,-1.0074113e+21,2.8538394e-23,3.7608065e+27,5.908254e+09,-1.0358884e+23,1172.6537,6.474672e-09,-2.5565433e-37,10.056378,-3.0833684e-32,3.0704527e-25,1.5737869e-25,0.00037047858,-2.9014565e-08,7.273278e-28,4.042097e-25,1.6760787e+10,-2.486916e-17,3.9128189e-25,4.6912477e+25,3.6349612e-28,-3.5500123e-32,1.2189488e-23,0.00022697076,3.036803e+22,-1.5892495e-22,-2.4989897e-20,4.169588e-14,6.5681565e+18,-9.5284095e-33,-3.4978757e-37,-1.5963062e-24,-7.825299e+13,-9.657747e+33,8.540422e-06,1.6688066e+25,4.5235946e+17,3.8409915e-19,-1.9024218,9.01061e+29,1.5857049e-36,-0.045959104,-0.48765832,219934.4,3.5202022e-06,-5.2572895e+16,1.06332814e-19,1.6878855e+21,1.1124703e+36,-0.0013628796,-0.008098686,1.1644483e+14,1.1506209e+27,-2.363297e+15,-8.797366e+29,-3.1646515e+10,2.7302717e+37,180914.69,4.003654e+35,1.6287233e-06,-6.5350525e-10,-9.9595354e-14,1.9082142e-28,-8.502033e+08,-1.2913452e-37,-1.397542e+31,-6.9751927e+27,-1.9940616e-31,-3.6061632e+06,-1.4330273e-37,-4.0765876e-06,-2.1710612e-34,3.4129258e+16,1.916916e-31,-7.31945e+12,-3.6976226e-07,6.549189e-12,-1.7604099e-22,6.7032794e-38,1.3270855e+37,0.09115147,-5.5887433e+16,1.9382685e-06,9.7065124e+23,2.3983637e+23,6.2179015e+34,-7.0738197e+16,-5.8958962e-21,-0.0013173277,1.5325367e-10,-1.686472e+29,-5.981692e+27,2.134029e-38,-300.63513,1.5182167e+09,-5.497695e-25,-3.515724e+06,-1.0229079e-10,7.669619e+08,-8.950921e-24,-2.0606466e+18,-6.672346e-21,-8.77055e+28,2.0106916e+32,-4.3676166e-32,3.706836e+06,-2.7567689e+29,2.259988e-34,-1.4178289e+22,2.1442774e-37,3.9487655e+06,-1.7616584e+28,-3.4006382e+31,-4.155656e+25,1.890566e+06,-3.257458e+18,2.4510466e+15,0.017132686,-3.2346718e-06,1.7831322e-22,-4.9496924e-05,-1.5901097e+16,-3.2673985e+06,-8.2441695e+18,1.7025943e-30,1.1662866e-20,-15.843527,-7.677223e+27,-3.0558472e-24,9.186566e+16,-6.0259187e-33,1.6686077e-24,-3.6429244e+35,1.0979995e-31,-3.0596927e+22,6555.556,-2.6153603e+38,-9.902111e-12,-0.0063768555,6.010126e-07,7.1446516e-06,-2.0227909e+25,4.145026e+35,79109.52,0.45703775,-2.1632778e+23,-37.810062,2.8897164e+38,-4.7258246e-22,-1.065596e-32,-1.9168973,6.075873e-27,-5.248355e+29,2.2442728e-29,-2.1488737e+20,9.596702e-13,2.1738736e-25,2.9303956e+28,1.3223075e+10,361.30347,-1.3720495e+33,7.883941e-36,-1.2344607e-15,6.506365e+28,-3.7322025e-25,-2.2768e+21,3.094598e-18,7.8778947e+18,-6.886267e-25,3.1071515e-19,-9.878897,1.0318615e+19,-3.840248e-19,-1884.953,2.1392393e-29,473.0001,2.3961792e+15,-8.556548e-10,-1.3132292e+21,-2.0156e+08,5.531843e+11,2.7666206e-29,3.0151126e+29,-1.1743612e-05,5.2001424e-38,-1.7267825e-32,-2.2218655e-29,4.795908e-26,4.26322e-33,-4.4808247e+23,2.9491661e+16,7.0634156e+37,8.850998e+37,4.7441448e-24,-0.20590198,-1.6532041e-19,2.8570024e-18,20.465265,-0.5828059,1.06081995e+18,2.1116827e+13,8.02485e-37,1.474554e+38,4.681161e-39,0.12359629,7.2575406e+29,-4.2761966e-27,-6.78153e-22,4.1253116e-12,-4.441542e+10,5.876969e-19,3.1587379e-25,1.7644251e+10,-0.10455486,1.08474665e+21,-4.8528612e-09,6.0334866e-09,-9.859063e+09,-0.0005710966,-0.53541034,-2.9818593e-28,2.5000448e-24,-5.2089874e-11,3.628935e+14,-3.4194535e+32,-6.8614863e+27,-620330.7,182.48586,-1897.718,2.4855858e-36,-3.0313352e-32,2.7838847e-30,2.1949982e+26,-1.0108186e+14,1.1618832e+06,-5.579587e-29,-3.6570487e+27,1.2861277e+21,-2.9072221e+32,4.6100554e+32,5.5429978e-06,-2.099898e+33,8.2342966e-38,-5.3309175e+06,-8.436482e-36,11180.096,80.66658,-2.790278e-19,6.688621e+21,-6.20712e-29,-1.602216e+32,2.0772916e+16,-1.2128008e-07,1.9517306e-15,306740.5,6.706911e-28,-9.4856576e+13,-7.2969403,9.3697063e-14,-2.7284395e-08,-0.51219296,9.933043e+25,6.929816e+06,2.8035113e+26,-4.622804e-12,2.8877507e-06,-4.4623687e-07,5.9159274e-34,-9.664706e+33,-6.7932725,-3.8727488e+09,0.07195243,-1.4311509e+23,-8.585135e+14,9.920355e+08,107911.555,-8.175896e+29,-2.3926886e-25,-3.911558e+22,1.4531091e+10,-1.4820188e-24,-3.1433295e+14,-2.1049373e-33,8.336927e-15,-411.7987,1.3010343e+25,-0.0003313821,1.7444015e-12,-7.0768507e-20,12955.507,-2.6363827e-30,6.7071592e+16,4.0032212e+31,-5.8123506e-38,3.501257e+34,-1.5768333e-26,571664.94,-1.0875803e+13,-5.765906e-24,5.8583213e-24,-1.3502036e-28,-3.978282e-19,-1.1967129e+38,-1.0019443e-25,2.794227e-25,1.4766238e+21,-7.8006707e+31,4.906036e+36,7.1526365e-24,-4.9748284e+37,1.5309933e-27,1.7580866e-18,-3.2069515e-23,9.276585e-20,0.02085023,5.660247e+25,-500.96536,-1.0790446e+25,3.2387402e-37,7.0705223e-34,415.4431,-1.5491291e+25,-1.5209308e+32,-1.7639003e-36,-2.312385e-31,915.83844,3.4827136e+23,-2.2685893e-21,1.2123725e-12,1.4067263e+35,-2.7106682e-30,-8.160034e-19,-8.269099e-14,-2.0501846e+29,-6.440597e+35,-27198.857,1.1916968e-18,-1.3580448e-14,-80203.86,5.4067123e+16,1.002248e-29,-8.235999e+21,-2.1319931e+32,-9.6433795e+25,-0.0050508147,6.876183e-12,-2.1448037e+11,1.0932636e-35,-2.5206542e+20,1.1379755e-18,3.3473056e+08,-1.3801489e-21,-5.922302e+29,-2.2384183e-11,2.3813358e+16,2.191137e+11,2.7811317e+32,-4.170178e-16,-4.157266e+12,2.361942e-28,3.3293865e+30,4.6952618e-08,2.8299622e+23,-2.8902507e-23,-3.6082216e-37,3.976342e-28,-2.5176647e-09,-1.364474e-19,-4.853335e-16,-1.4068841e+26,-2.7112489e-37,-4.6311895e-29,6.6290147e-07,4.5698465e+19,-9.0849206e-10,2.1352787e+08,-2.351589,-1.2410396e+14,1.0074773e+11,-4.6581555e-27,-1.7445071e-24,3.6160867e+22,-2.3940403e-08,4.8409666e+12,-1.8118821e+25,302.84012,-9.764049e+30,-3.43489e+37,-8.929037e-30,8.659946e+35,-7.718583e+13,2.5115726e+19,5.4481886e-34,5.301091e+36,-2.6222505e-30,-0.52291447,7.950792e-08,-1.982578e-12,-6.191117e-16,1.0003973e-29,1.5578165e-09,7.1802395e+24,-1.7309309e+22,1.9581703e+32,1.8933647e+24,1.2304442e+34,2.1141041e-32,-7.924158e+30,1.3529993e-38,8.658044e+21,-2.0404357e-38,-1.171119e+10,-7.6508765e+34,-8.812689e+25,2.8141328e-26,-0.319516,5.6074894e-33,-1.11386726e-07,2.3228779e+34,-1.1007954e-24,-5.7135096e+10,7.2152436e+23,-1.2422854e+17,2.009747e-26,1.3617161e-17,-2.6414745e+38,-1.3612453e-05,3.6156828e+27,-2.3362963e+20,1.1746824e+19,-4.598563e-20,2.142123e-07,-1.4319363e+07,8.773403e-20,2.1635209e+15,1.5730275e+08,1.3624456e-17,-1.7652465e-09,0.0080164205,-8.396407e+18,-16755.973,-1.3306829e-08,1.2146457e+12,-3.1069982e-36,2.1676271e-23,-7.9833915e-21,1.1286395e+21,-3.0401037e+29,1.2326575e+18,3.6082045e+10,8.833559e-09,1.5275371e+35,-2.3070097e-31,-2.8087025e-33,-7.264299e+06,-5.7377576e+21,6.13361e+16,1.1588772e-18,1.9813485e+17,-6.375514e+13,-4.5895186e-24,-48.872242,-1.9752131e-33,-1.1583094e+21,-1.6944044e+29,1.4562444e-28,1.1787453e+37,7.9364065e-37,-1.2421236e+13,-8.415504e-12,8.2693344e-36,1.5106983e-19,7.813769e-16,1.2802598e+30,-4.214806e+35,0.00023199985,1.6879921e-32,-7.0191073e+30,-5.904207e+14,1.6581321e+06,3.0973828e+15,6470.786,4.2728628e-15,-141.49622,1.054999e-23,-1.2135518e+28,3.1141637e-26,-1.7171658e-29,8.2708645e-17,7.583206e+36,-0.00026330433,-9.196725e+16,5.4894365e-24,1.4768997e+37,-7.0779444e+20,8.6175826e-13,-5.9129772e-05,7.18584e+34,4.5120277e-08,-9.582516e+12,1.01875235e+37,3.888929e-09,5.485976e-21,14.21636,-1.4705464e+11,1.5432105e+32,5.1872775e-13,-5.655799e-33,3.1215332e-16,5.3987824e+26,-2.88211e-39,-7.4834166e-21,6.5213498e-21,-8.53204e-34,-4.7960187e-15,2.7011638e+29,1.35874896e+11,-1.411362e-25,13.454742,-4.004447e+13,1.3900874e-05,5.349957e+37,6.645692e-14,2.9366022e-13,-3.3648097e+16,746185.4,6.793217e+12,1.6638164e-33,3.572226e-33,2.2222912e+15,-1.7219907e-27,-1.6628906e+26,1.2779025e-08,1.1090537e+07,-2.3310674e+32,-2.8802376e-34,-1.6974874e+17,-3.7897673,-6.5825304e-18,3.522349e+37,0.0003127239,2.375761e+06,-1.4628257e+22,0.0432303,1.1480656e+08,3.2502333e-35,-1.9551248e+31,-12365.706,1.7793183e+16,6.321664e-35,6.862961e-38,-6.6204984e-12,-7.3744755e+29,-1.1126302e-35,-1.8013315e+38,-3.9098946e+14,-1.9361735e+17,-111.53434,3.4261977e-10,-2.796669e+09,9.638301e+15,1.2704937e+35,5.3806655e+20,6097.1313,2.1060073e+33,-4.8217065e+20,-2.920228e+07,-1.2322693e-24,4.7929525e-26,2.2116308e+14,-9.878864e+25,5.8268394e-37,1.077446e-36,-8.273371e-06,-1.9061532e+31,-1.3462625e+06,-0.00029358058,-2.9599218e-31,-3.248571e+31,-8.2019255e-20,5.6169686e+20,-3.259711e+09,-1.5605991e+35,-6.9308392e-37,6.7261343e-21,-5.782721e+36,-1.2677762e+29,2.8424804e+10,-6.205873e-30,-1.0901324e-14,-5.0379116e+09,-1.7784757e+13,-1.8120162e+28,-1.230605e-07,9.002646e-33,3.0408723e-06,-1.5619607e-33,-3.6651414e-14,1.3841812e+37,5.8408812e-05,-5.544728e+10,6.663682e-28,4.2473222e+25,-12.390379,2.8231353e-08,1.3396806e+27,-2.75007e+11,2.6270658e-26,1.4055541e-07,1.1089125e+30,-2.0230576e-30,1.349922e-31,-4.885793e-31,-3.9327774e-28,1.111431e-16,-6.638077e-39,4.0699318e-38,-1.2473148e+11,4.254183e+23,-9.0740465e-36,1.3377762e-24,-1.7266462e+34,3.202615e+30,-1.3170641e+30,-9.068939e-08,3.8961834e+16,-1.338175,-1.3313016e+23,4.4450883e+26,-9.158098e-11,1.83133e-32,1.1024674e+29,-9.43901e-10,-1.8849181e-22,-4.038383e-28,-4.4963087e+29,1.6531504e-21,9.784689e-23,-5.2275456e+24,-3.4437886e-14,-1.9392433e+14,1.1475589e-29,90950.35,-1.6832255e-17,1.4920932e+10,-8.0461364e-16,1.2207706e+22,7.039735e+27,1.5200758e-31]':::VECTOR,ARRAY['36 years 6 mons 682 days 01:39:09.280398':::INTERVAL,'-69 years -4 mons -1 days -07:24:03.755765':::INTERVAL,'1 mon 1 day':::INTERVAL,'-73 years -11 mons -911 days -20:44:03.770623':::INTERVAL,'26 years 557 days 05:43:59.522922':::INTERVAL],25:::OID,B'1001011001101110011011011010100111100001001101100','/'::BPCHAR,'\\x96e7f8e0':::BYTES,'[[true, null, true], false, 1.032756339380738]':::JSONB),('2031-01-22 14:54:46.00067+00':::TIMESTAMPTZ,false,'[-1.6623596e-16,-2.5801049e-11,0.00072758214,3.295117e-10,3.6419915e+35,-1.4826973e-24,-4.6643613e-35,2.475906e+16,-2.940861e+22,1.18704525e+27,4.152039e-34,7.5851306e-32,-1.8827746e-33,-4.049218e+24,1.0721411e-22,-3.0809838e-12,-1.29261215e-08,-1.4784033e+17,7.349292e+16,-1.3378628e-14,-1.2411375e+22,-1.3683929e-07,2.1668315e+11,4.5166386e-09,-0.18504189,8.691682e-32,-1.2091455e+15,-5.004633e+20,-7.1962636e-09,-3.845603e-37,1.0729571e-11,-8.6767174e+18,1.1006917e-16,5.7570748e+29,1.3272438e+10,-2.3869708e-27,-0.00019203655,3.6821245e+06,1.1590688e+25,-7.200846,-1.4141741e+15,1.1222165e-09,1.2243506e+19,-6.636352e+24,-2.7656565e+06,-3.1064774e-05,-4.6077902e-20,-4.4231055e-29,-6.123408e-07,7.1423776e+20,-2.1080594e-27,-2.2449837e+12,-3.207294e+07,1.7997912e+09,1.0497032e-06,-3.7068904e-10,-2.948282e+15,1.04252934e+09,1.2191185e-38,2.8310424e+13,-4.506733e+35,-9.411541e-39,-60.161396,2.5558336e+08,114.41861,1.13562935e+20,-51850.73,-3.5966068e-27,4.627158e-08,1.7913953e-28,1.4352423e-06,-71030.32,-8.7137865e+11,1.3129327e-18,-118388.78,-6.2249523e+34,-8.1673164e-07,4.924721e-12,-1.1920881e+30,-1.6517624e-14,6.8660085e+06,-3.851097e-32,3.442909e+16,-5.1724515,-1.3833773e+20,-3.5677207e+28,-2.153303e+08,-2.6204897e+23,8.495888e-29,2.6443065e-30,1.0556043e+28,1.5903603e-21,500.46274,-1.7790227e-36,-0.028329257,-4.930537e-05,6.4324865e-08,1.04614216e+37,-6.0622053e-13,982.7813,-7.7511464e-29,-1.5055395e+29,-1.3968812e-32,5.4632606e-23,1.6933578e-16,-0.02637448,8.696192e-18,-0.030546436,1.4888363e-15,2.3918282e+06,8.7083665e-38,-2.215464e+29,8.843047e-06,1.9075982e+28,9.0492435e+18,-2.0268964e+24,-4.9367435e+14,1.2379644e+23,-5.083821e+34,5.475123e-12,1.515077e-13,-3.8031974,-0.0038159129,2.4936552e+14,4.168045e+34,-1.4187848e+23,-6.458824e-17,1.4298384e+29,1.4928875e+18,6.9275076e-27,1.5833466e-27,1.7461829e+14,-2.2447733e-17,9.103488e+33,-3.199925e+14,-0.0011667731,-6.7544784e-16,-1.8975333e-25,2.4451542e-24,5.6315726e+23,-778645.7,-4.0724992e-19,-1.8317505e+12,1.423376e-26,8.93797e-14,2.8283523e+28,-3.5759256e-35,6.277098e-23,5.78272e+20,-1.1727818e-37,1.596757e-37,-1.4022412e-13,-5.781806e-05,-3.8161317e+27,-1.9313361e-37,-0.00017042909,2.6749496e+38,-5.294434e-18,3.230168e+30,-9.6752956e+20,-0.028011544,4.7328332e+14,-1.352576e-32,-1.6518983e+35,2.3105256e-13,-1.8350457e+28,-6.7847197e+10,4.387295e-17,3.1678786e-28,2.4726307e-05,38.160694,-3.7060145e-06,7.819626e-08,1.2077854e+24,-2.119548e+22,229.5507,1.7329576e+12,1667.0159,920725.25,-1.848726e-07,9.2765195e-18,-2.5313092e-28,2.8765764e-22,-1.663638e-09,-2.4368238e+38,7.210309e-32,-2.1825968e+07,-8.2032925e-32,1.0836859e+20,-1.1991002e-18,3.8541665e+21,-239245.3,-6.3977053e+37,-8.936936e-31,491770.28,6.677479e+30,-5.381144e+10,1.0763373e-25,-4.292043e-06,4.964525e-37,3.361268e-17,1.1286894e+14,1.2181895e-35,-870.6743,0.018975252,4.4526613e-29,2.0325845e+13,-1.6972336e+37,5.3284857e-17,-3.872633e-24,6.211085e+24,-1.85177e+19,-6.5166947e+13,1.213605e-14,-1.5718403e+07,-413567.62,7.890114e+09,26.935623,-1246.859,3.9190704e-24,57.360542,4.6162145e+06,3.4464855e-19,-8.736136e+15,-4.33065e-30,793.22095,-2.3658227e-23,-0.0014828084,3.401617e+30,0.106904216,5.615953e-20,2.7776131e-27,9.7024046e-24,-7.982996e-21,3.4707244e+09,-261.0699,-1.4765808e-29,-1.2353211e-24,77638.22,-1.848569e-38,-5.9964684e-27,0.20199062,2.3087456e+26,-5.740452e-13,-4.838937e-37,-3.8014352e-31,3.0353627e+27,3.8741906e-24,-4.479201e+09,0.00015530612,1.0033301e-35,-1.5367813e+32,2.6761569e-08,1.5897442e+14,-3.045253e+27,-1.3551924e+23,-4.899347e+36,8.9771214e+17,4.451816e-19,4.83389e+20,7.809647e-11,-0.0022639604,-7.1564503e-38,4.3734394e+17,1.14860395e+36,-2.9514025e-29,1.1984802e-11,0.00026780638,0.00018963276,-3.0660605e+25,7.8553007e-35,2.3408524e-34,1.9496152e+06,1.2935566e-18,0.011211588,-6.3799503e+34,4.1300956e+21,1.053296e-15,0.14832333,-7.8721367e-26,-5.1625855e-32,2.7963395e+08,3.0072838e+10,1.366177e+37,2.240399e+31,3.3075246e-38,-1.4095081e+31,1.2427505e-30,-6.7113194e-12,1.621612e-25,5.0776368e+17,1.2580868e-12,-1.4959875e-18,-1.5602744e+09,-648311.56,9.363547e-29,0.0040318766,3.4203603e+30,-2.830677e-36,2.225863e+29,-4.048603e-08,7.447235e-24,8.053636e+18,32.94808,1.7339981e-37,4.26145e-17,1.1374538e-14,-0.06284237,8.604604e+31,3.0552025e-24,-2.809242e+25,-3.3803257e-21,-1.4862264e-31,-1.9465363e+34,-3.471492e-33,-2.7274663e-12,1.3172563e-31,8.069656e+35,-4.2350983e+31,6.468151e+37,1.248958e+14,1.2937732e+24,-1.2537763e-06,1.1061911e-24,-8.21355e+27,4.2161025e+25,-2.9582083e-09,4.5264625e-16,6.552505e-31,1.5454883e-13,-1.3253492e-38,3.1167359e+35,-1.0501683e+21,-1.7741845e-36,1.2856291e+07,-4.8630486e-16,-2.283206e-14,2.1229595e+22,-0.02990471,-4.7243734e-21,2.728835e-12,-3.5043924e+32,-5.5909393e-23,2.4273007e-24,-1.7595768e-05,-1.9403698e+22,-2.1984843e-20,-1.5614545e+07,-1.0258174e+09,1.1035159e+21,6.436094e-09,2.573139e-28,-2.063548e+09,-0.012866648,-2.2034294e-35,-1.1708103e-26,-1.3946688e+07,-1.384396e+12,-30311,4.9037497e-25,1.1096089,8.884517e+13,1.7916989e-06,2.181767e-09,2.8073034e-06,8.793971e-09,-9.1790396e-24,7.8755704e+07,-6.3732635e+17,-7.010227e-10,3.9904376e-06,-7.748955e-35,9.099867e+17,1.5178809e-30,-1.5108781e+11,-1.961348e-06,-5.4138444e+11,-1.7010716e-20,-1.904569e+29,-1.3942971e+34,0.00045679495,9.683905e+14,2.1273311e+24,208.52606,-46.97977,-1.9303656e-12,-1.411999e-23,-8.69292e+24,8.538812e-31,383.57104,-1.4232193e+23,-5.493484e-25,5.9968195e+06,-2.1460622e-08,3.0260817e+24,1.5083255e-12,-4.9712086e-14,8.6358556e-30,2.7703864e-15,4.325476e+35,-2.8896446e-15,-1.1187967e+17,0.0775689,0.078226216,8.2595815e-09,-2.57681e+27,-8.709836e+07,-8.9778115e+19,-693.263,-2.54321e+38,5.403087e-14,5.8403183e-37,-1.0445655e-23,1.167904e-26,-1.3594094e-25,8.36929e+16,-4.0537954e+29,-5.6762186e+25,2.6762975e-28,1.7176617e-36,3.2251477e+12,-1.986542e-12,2.7139515e+23,1.0813329,-2.5236842e+33,-0.3111691,-1.0840968e+20,402747.56,4.4318637e-27,-2.2691697e-35,-5.511167e-17,2.9650363e-20,-9.909601e+11,52.52521,1.2245341e+14,8.774238e+20,-4.686173e+13,-4.3548353e+27,8.974913e-21,-2.7688653e-19,7.02154e+37,-5.413355e-32,-1.9104611e-23,-3.2376407e-34,1.1389987e-11,5.0291892e-26,5.1817445e-08,4.2849037e+36,1.8549749e+15,-5.2185627e-13,2.2174308e-10,8.758939e+07,-3.0258107e-05,-1.2801884e+16,-6.8193316e-36,-6.204513e+22,7.6938898e+25,-9.530894e-32,2.8069012e+26,-7.7590536e+27,4.625389e-10,-2.6702106e-22,-4.4574186e+23,-2.735952e+23,-5.2893383e+35,1.8428947e+37,-2.0255947e-23,6.035553e-38,9.141302e+34,-9.172346e+22,-1.1202612e-31,-1.4571213e+30,-6.99933e+14,1.5712596e+20,0.006274708,2.3009029e+08,-1.1363701e+23,-8.405775e-31,-1.6871062e-30,-1.299012e+09,5.98769e-06,2.3854797e-37,1.0420648e-13,7.66766e-28,123.79786,-5.677355e+35,-6.973152e+19,-9.340217e+36,-12.692927,-8.442796e+09,-2.7257911e+10,-3.1600547e-16,-3.8261555e+12,8.728403e+28,-5.0051804e-19,-6.020302e+16,8.762349e+19,-4.4493638e-13,3.5107817e-20,-3.1217005e+11,7.771179e+25,-4.5953236e-10,1.085413e-28,-40657.707,6.2777874e+31,1.7481258e+18,6.571508e-07,1.6951117e-10,0.21790215,-3.4333004e-18,-1.2025259e-05,2.9543312e-09,1.9046385e-10,-1.3878202e+10,1.3768598e+17,-159125.9,-7.68893e-21,3.2730543e+32,1.4864939e-25,-1.22709975e+29,-8.583615e+18,-1.9513912e-38,-2.671584e-14,3340.8088,-3.6203702e+15,-0.002777583,2.9753141e+16,-3.384937e+30,5.105792e-36,9.0254e+25,5.239276e+08,5.381375e-26,5.18944e-14,-1.8500761e-22,-6.038021e+28,4.096794e+27,-0.0049080076,2.4221373,-6.7388227e+34,1.5631829e-14,-1.1980247e-23,-1.3835209e+21,2.4616495e-16,-5.2588635e+20,-3.816363e-34,1.0140472e+09,7.5013765e+31,-1.5430321e+06,6.2966605e+22,7.2525257e+12,-2.7567447e+14,-1.2250308e-27,453744.78,-1.4647944e-38,-1.6891787e-05,-2.2171994e-14,2.1227465e+21,-3.2683964e+12,1.2930309e-32,-1.5853976e+13,9.1300666e+20,-2.1351219e-15,4.8189926e+34,-2.4931493e+37,3.0020421e-24,-2.727399e+33,-0.00022905384,1.382958e-35,-5.3030913e-12,-1.7798255e-31,9.713736e+32,-4.011359e+21,3.0557109e-18,0.0016289062,3.456194e+25,27015.7,3.2774963e-27,5.2827175e-17,-1.0820451e-17,7.808343e+34,-2.4424515e+27,28.748446,2.908717e-05,1.3237166e-19,-4.9655098e+29,-8.30884e+32,-1.1960483e-38,-8.599991e+24,-8.540569e-17,-1.4353346e-05,3.0533886e+35,2.9020595e+08,1.0941428e-16,-5.7736617e+37,7.688865e-32,-3.7860483e+23,-6.979398e+32,-2.5848459e+36,-3.3483312e+37,6.4225227e+19,599321.4,-3.294214e+12,-5.58398e+28,-3.2069302e-13,-1.0063943e-32,-6651.9663,-3.2639142e+29,2.053993e+12,-2.5896314e+21,-9.946378e-27,1.5409447e-05,1.1318779e-06,-2.1423292e-08,1.2492155e+10,2.7443388e+38,-4.1677976e-36,-1.2310873,-2.4809041e+29,6.5518642e-15,-9.1559656e-08,-1.8601428e+25,4.9326448e-30,-1.3813777e+22,-6.4373134e+27,2.2075119e-13,2.889721e+22,4.4281234e+25,256852.95,4.1984113e-21,8.85643e-36,-8.006884e-36,6.760344e-18,1.9580125e-24,5.9759564e-05,-2.1887176e+24,-1.4229582e-07,4.5960352e+21,1.0721138e-28,-0.1712409,-0.062192623,1.2010424e-11,253.81096,-1.2828883e+29,-4.6906843e+10,-14557.383,-1.8515112e+09,-3.0817035e-13,-3.7273668e+23,9.66332e-09,-1.5672591e-34,-0.000742539,-6.33574e-10,-3.5375928e+15,-7.119992e+24,1.7683655e-21,-2.0417842e-16,1.5594975e+13,-9.65436e-40,7.474836e+22,2.2489014e-05,151.43892,-2.211638e+31,-0.5964576,-9.596643e+22,1.2304951e+07,8.084741e-34,1.8981006e+21,-2.6540272e-12,5.04192e+26,3.668165e+17,5.212423e+25,-6.6318766e+16,2.3948377e+28,6.5986694e+28,5.684523e+29,2.2027418e+09,1.9506538e+24,-2.6812603e-07,-1.0207196e-19,1.6272874e-37,-1.1443402e-31,-2.070031e+34,-7.442582e+18,4.438497e-10,-1.7987093e-32,-0.030774506,-1.4401182e+20,-5.2094513e+23,1.997077e+07,-4.002643e-13,-6.085049e+30,-9.851271e+30,9.289125e-19,-7.3759944e-28,3.8389935e+33,4.717381e+08,3.815912e+22,-8.086609e+16,7.905664e-17,-1.1534158e+08,-2.0352438e+10,0.18759885,-5.994288e+14,-4.4955434e-22,9.660948e+37,-7.5154973e-35,-1.5419129e+24,1.9058272e+11,1.0811084e+20,3.4491188e-34,236.76219,1.1326765e+18,-8.257325e-19,276019.47,-4.479137,-1.2517884e+11,1.6501786e-10,1.6174534e-35,7.3416842e+25,-1.1899571e+26,-2.4563456e-05,-1.4735637e-27,1.5972144e-34,-1.4481068e-08,-2.2479905e-09,9.2747256e-35,-1.7063362e-10,-3.42819e-16,4.3133747e-16,-4.025684e+21,2.2171402e+34,5.8125115e+10,-6.9892185e-28,-9.7106526e-20]':::VECTOR,ARRAY['-67 years -3 mons -203 days -18:16:22.18931':::INTERVAL,'-45 years -11 mons -574 days -15:51:27.25659':::INTERVAL,'-43 years -11 mons -282 days -06:06:55.734429':::INTERVAL,'-49 years -3 mons -927 days -23:17:18.40906':::INTERVAL,'4 years 9 mons 140 days 23:45:53.940871':::INTERVAL],17:::OID,B'1011110111010100011010001101010010111101111100100',e'\\x1e'::BPCHAR,'\\x':::BYTES,'[true, null, false]':::JSONB),('2005-07-22 02:39:35.000082+00':::TIMESTAMPTZ,false,'[-1.7956231e-14,-1.242985e-18]':::VECTOR,ARRAY['-80 years -10 mons -862 days -17:00:17.330992':::INTERVAL,'75 years 1 mon 538 days 02:08:35.732572':::INTERVAL,'-7 years -5 mons -842 days -14:44:48.13714':::INTERVAL,'-80 years -7 mons -278 days -19:54:26.062529':::INTERVAL],90004:::OID,B'0111101010111100011101110111101111011011000100101','G'::BPCHAR,'\\xc35fe88abebb76':::BYTES,'{\"test\": \"json\"}':::JSONB)",
   "potentialExecErr": "23502,23505,23514"
  },
  "COMMIT"
 ],
 "expectedExecErrors": "",
 "expectedCommitErrors": "",
 "message": ""
}
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=15</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestWorkload.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>


Jira issue: CRDB-42933

pkg/ccl/testccl/workload/schemachange/schemachange_test.TestWorkload [failed](https://tanzanite.cluster.engflow.com/invocations/default/d142a562-8acb-4626-81f5-4279b440ae8e?testReportRun=2&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL3Rlc3RjY2wvd29ya2xvYWQvc2NoZW1hY2hhbmdlOnNjaGVtYWNoYW5nZV90ZXN0) on master @ [645eb8c99796b3b88f5631aa0fc92a011010ce64](https://github.com/cockroachdb/cockroach/commits/645eb8c99796b3b88f5631aa0fc92a011010ce64):


```
  },
  "COMMIT"
 ],
 "expectedExecErrors": "",
 "expectedCommitErrors": "",
 "message": ""
}
    schema_change_external_test.go:131: 
        	Error Trace:	pkg/ccl/testccl/workload/schemachange/schemachange_test_test/pkg/ccl/testccl/workload/schemachange/schema_change_external_test.go:131
        	Error:      	Received unexpected error:
        	            	***UNEXPECTED ERROR; Failed to generate a random operation: failed generating operation: insertRow: ERROR: column "crdb_internal_column_3_name_placeholder" does not exist, referenced in "crdb_internal_column_3_name_placeholder IS NOT NULL" (SQLSTATE 42703)
        	            	(1) forced error mark
        	            	  | "fatal error when running txn"
        	            	  | github.com/cockroachdb/errors/withstack/*withstack.withStack::
        	            	Wraps: (2)
        	            	Wraps: (3) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*schemaChangeWorker).runInTxn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/workload/schemachange/schemachange.go:486
        	            	  | [...repeated from below...]
        	            	Wraps: (4) ***UNEXPECTED ERROR; Failed to generate a random operation
        	            	Wraps: (5) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*operationGenerator).randOp
        	            	  | 	github.com/cockroachdb/cockroach/pkg/workload/schemachange/operation_generator.go:219
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*schemaChangeWorker).runInTxn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/workload/schemachange/schemachange.go:473
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*schemaChangeWorker).run
        	            	  | 	github.com/cockroachdb/cockroach/pkg/workload/schemachange/schemachange.go:592
        	            	  | pkg/ccl/testccl/workload/schemachange/schemachange_test_test.TestWorkload.TestWorkload.func4.func6
        	            	  | 	pkg/ccl/testccl/workload/schemachange/schemachange_test_test/pkg/ccl/testccl/workload/schemachange/schema_change_external_test.go:120
        	            	  | golang.org/x/sync/errgroup.(*Group).Go.func1
        	            	  | 	golang.org/x/sync/errgroup/external/org_golang_x_sync/errgroup/errgroup.go:78
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1695
        	            	Wraps: (6) failed generating operation: insertRow
        	            	Wraps: (7) ERROR: column "crdb_internal_column_3_name_placeholder" does not exist, referenced in "crdb_internal_column_3_name_placeholder IS NOT NULL" (SQLSTATE 42703)
        	            	Error types: (1) *markers.withMark (2) *schemachange.ErrorState (3) *withstack.withStack (4) *errutil.withPrefix (5) *withstack.withStack (6) *errutil.withPrefix (7) *pgconn.PgError
        	Test:       	TestWorkload
Schema Workload Stats
Total Schema Statements Executed = 33
Total Schema Statements Succeeded = 33
Total Schema Statement Expected Failures = 0
Total Transactions Committed = 24
Total Transactions Rolled Back = 18
Total Transactions Executed = 42
    schema_change_external_test.go:96: backup, tracing data, and system table dumps in /var/lib/engflow/worker/work/1/exec/bazel-out/k8-fastbuild/testlogs/pkg/ccl/testccl/workload/schemachange/schemachange_test/run_2_of_25/test.outputs/logTestWorkload299870959
    panic.go:626: -- test log scope end --
test logs left over in: outputs.zip/logTestWorkload299870959
--- FAIL: TestWorkload (13.34s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=2</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestWorkload.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

pkg/ccl/testccl/workload/schemachange/schemachange_test.TestWorkload [failed](https://tanzanite.cluster.engflow.com/invocations/default/7bc28c7b-f7d6-4dd4-96a9-0dbd249c3699?testReportRun=12&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL3Rlc3RjY2wvd29ya2xvYWQvc2NoZW1hY2hhbmdlOnNjaGVtYWNoYW5nZV90ZXN0) on master @ [49ca24cedb042579e9645c206640d59975805d12](https://github.com/cockroachdb/cockroach/commits/49ca24cedb042579e9645c206640d59975805d12):


```
    ],
    "result": [
     "abs(col14_w1_16)",
     "abs(col14_w1_16)"
    ]
   }
  ],
  "previousStatements": [
   "ALTER TABLE public.table_w1_14 DROP COLUMN col14_w1_19"
  ],
  "usesDeclarativeSchemaChanger": true
 }
}
    schema_change_external_test.go:131: 
        	Error Trace:	pkg/ccl/testccl/workload/schemachange/schemachange_test_test/pkg/ccl/testccl/workload/schemachange/schema_change_external_test.go:131
        	Error:      	Received unexpected error:
        	            	***UNEXPECTED ERROR; Received an unexpected execution error.: ERROR: column "col14_w1_19" is referenced by the primary key (SQLSTATE 42P10)
        	            	(1) forced error mark
        	            	  | "fatal error when running txn"
        	            	  | github.com/cockroachdb/errors/withstack/*withstack.withStack::
        	            	Wraps: (2)
        	            	Wraps: (3) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*opStmt).executeStmt
        	            	  | 	github.com/cockroachdb/cockroach/pkg/workload/schemachange/operation_generator.go:3188
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*schemaChangeWorker).runInTxn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/workload/schemachange/schemachange.go:495
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*schemaChangeWorker).run
        	            	  | 	github.com/cockroachdb/cockroach/pkg/workload/schemachange/schemachange.go:592
        	            	  | pkg/ccl/testccl/workload/schemachange/schemachange_test_test.TestWorkload.TestWorkload.func4.func6
        	            	  | 	pkg/ccl/testccl/workload/schemachange/schemachange_test_test/pkg/ccl/testccl/workload/schemachange/schema_change_external_test.go:120
        	            	  | golang.org/x/sync/errgroup.(*Group).Go.func1
        	            	  | 	golang.org/x/sync/errgroup/external/org_golang_x_sync/errgroup/errgroup.go:78
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1700
        	            	Wraps: (4) ***UNEXPECTED ERROR; Received an unexpected execution error.
        	            	Wraps: (5) ERROR: column "col14_w1_19" is referenced by the primary key (SQLSTATE 42P10)
        	            	Error types: (1) *markers.withMark (2) *schemachange.ErrorState (3) *withstack.withStack (4) *errutil.withPrefix (5) *pgconn.PgError
        	Test:       	TestWorkload
Schema Workload Stats
Total Schema Statements Executed = 22
Total Schema Statements Succeeded = 22
Total Schema Statement Expected Failures = 0
Total Transactions Committed = 11
Total Transactions Rolled Back = 18
Total Transactions Executed = 29
    schema_change_external_test.go:96: backup, tracing data, and system table dumps in /var/lib/engflow/worker/work/1/exec/bazel-out/k8-fastbuild/testlogs/pkg/ccl/testccl/workload/schemachange/schemachange_test/run_12_of_25/test.outputs/logTestWorkload1711528738
    panic.go:629: -- test log scope end --
test logs left over in: outputs.zip/logTestWorkload1711528738
--- FAIL: TestWorkload (9.37s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=12</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestWorkload.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

pkg/ccl/testccl/workload/schemachange/schemachange_test.TestWorkload [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/17322448?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/17322448?buildTab=artifacts#/) on master @ [47b5992396988e25e4b0c4c99d8e39265a3e481f](https://github.com/cockroachdb/cockroach/commits/47b5992396988e25e4b0c4c99d8e39265a3e481f):


```
     "CASE WHEN col25_w1_27 IS NULL THEN e'@\\x18\\\\':::STRING ELSE e'\\x00':::STRING END",
     "lower(CAST(col25_w1_29 AS STRING))",
     "lower(CAST(col25_w1_29 AS STRING))",
     "abs(col25_w1_28)"
    ]
   }
  ],
  "previousStatements": [
   "ALTER TABLE public.table_w1_61 DROP COLUMN col25_w1_32"
  ],
  "usesDeclarativeSchemaChanger": true
 }
}
    schema_change_external_test.go:131: 
        	Error Trace:	pkg/ccl/testccl/workload/schemachange/schemachange_test_test/pkg/ccl/testccl/workload/schemachange/schema_change_external_test.go:131
        	Error:      	Received unexpected error:
        	            	***UNEXPECTED ERROR; Received an unexpected execution error.: ERROR: column "col25_w1_32" is referenced by the primary key (SQLSTATE 42P10)
        	            	(1) forced error mark
        	            	  | "fatal error when running txn"
        	            	  | github.com/cockroachdb/errors/withstack/*withstack.withStack::
        	            	Wraps: (2)
        	            	Wraps: (3) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*opStmt).executeStmt
        	            	  | 	github.com/cockroachdb/cockroach/pkg/workload/schemachange/operation_generator.go:3188
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*schemaChangeWorker).runInTxn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/workload/schemachange/schemachange.go:495
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*schemaChangeWorker).run
        	            	  | 	github.com/cockroachdb/cockroach/pkg/workload/schemachange/schemachange.go:592
        	            	  | pkg/ccl/testccl/workload/schemachange/schemachange_test_test.TestWorkload.TestWorkload.func4.func6
        	            	  | 	pkg/ccl/testccl/workload/schemachange/schemachange_test_test/pkg/ccl/testccl/workload/schemachange/schema_change_external_test.go:120
        	            	  | golang.org/x/sync/errgroup.(*Group).Go.func1
        	            	  | 	golang.org/x/sync/errgroup/external/org_golang_x_sync/errgroup/errgroup.go:78
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_arm64.s:1223
        	            	Wraps: (4) ***UNEXPECTED ERROR; Received an unexpected execution error.
        	            	Wraps: (5) ERROR: column "col25_w1_32" is referenced by the primary key (SQLSTATE 42P10)
        	            	Error types: (1) *markers.withMark (2) *schemachange.ErrorState (3) *withstack.withStack (4) *errutil.withPrefix (5) *pgconn.PgError
        	Test:       	TestWorkload
Schema Workload Stats
Total Schema Statements Executed = 25
Total Schema Statements Succeeded = 25
Total Schema Statement Expected Failures = 0
Total Transactions Committed = 17
Total Transactions Rolled Back = 11
Total Transactions Executed = 28
    schema_change_external_test.go:96: backup, tracing data, and system table dumps in /artifacts/tmp/_tmp/67537770b84c3a4ea177539c1e46bd43/logTestWorkload395502182
    panic.go:629: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/67537770b84c3a4ea177539c1e46bd43/logTestWorkload395502182
--- FAIL: TestWorkload (9.01s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestWorkload.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

pkg/ccl/testccl/workload/schemachange/schemachange_test.TestWorkload [failed](https://tanzanite.cluster.engflow.com/invocations/default/1e66a25e-65da-41f8-a943-300a3da4393d?testReportRun=13&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL3Rlc3RjY2wvd29ya2xvYWQvc2NoZW1hY2hhbmdlOnNjaGVtYWNoYW5nZV90ZXN0) on master @ [ebbe08756db4defbb6388041737e240ac07783a3](https://github.com/cockroachdb/cockroach/commits/ebbe08756db4defbb6388041737e240ac07783a3):


```
    ],
    "result": [
     "abs(col6_w0_14)",
     "abs(col6_w0_14)"
    ]
   }
  ],
  "previousStatements": [
   "ALTER TABLE public.table_w0_6 DROP COLUMN col6_w0_15"
  ],
  "usesDeclarativeSchemaChanger": true
 }
}
    schema_change_external_test.go:131: 
        	Error Trace:	pkg/ccl/testccl/workload/schemachange/schemachange_test_test/pkg/ccl/testccl/workload/schemachange/schema_change_external_test.go:131
        	Error:      	Received unexpected error:
        	            	***UNEXPECTED ERROR; Received an unexpected execution error.: ERROR: column "col6_w0_15" is referenced by the primary key (SQLSTATE 42P10)
        	            	(1) forced error mark
        	            	  | "fatal error when running txn"
        	            	  | github.com/cockroachdb/errors/withstack/*withstack.withStack::
        	            	Wraps: (2)
        	            	Wraps: (3) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*opStmt).executeStmt
        	            	  | 	github.com/cockroachdb/cockroach/pkg/workload/schemachange/operation_generator.go:3188
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*schemaChangeWorker).runInTxn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/workload/schemachange/schemachange.go:495
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*schemaChangeWorker).run
        	            	  | 	github.com/cockroachdb/cockroach/pkg/workload/schemachange/schemachange.go:592
        	            	  | pkg/ccl/testccl/workload/schemachange/schemachange_test_test.TestWorkload.TestWorkload.func4.func6
        	            	  | 	pkg/ccl/testccl/workload/schemachange/schemachange_test_test/pkg/ccl/testccl/workload/schemachange/schema_change_external_test.go:120
        	            	  | golang.org/x/sync/errgroup.(*Group).Go.func1
        	            	  | 	golang.org/x/sync/errgroup/external/org_golang_x_sync/errgroup/errgroup.go:78
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1700
        	            	Wraps: (4) ***UNEXPECTED ERROR; Received an unexpected execution error.
        	            	Wraps: (5) ERROR: column "col6_w0_15" is referenced by the primary key (SQLSTATE 42P10)
        	            	Error types: (1) *markers.withMark (2) *schemachange.ErrorState (3) *withstack.withStack (4) *errutil.withPrefix (5) *pgconn.PgError
        	Test:       	TestWorkload
Schema Workload Stats
Total Schema Statements Executed = 22
Total Schema Statements Succeeded = 22
Total Schema Statement Expected Failures = 0
Total Transactions Committed = 10
Total Transactions Rolled Back = 10
Total Transactions Executed = 20
    schema_change_external_test.go:96: backup, tracing data, and system table dumps in /var/lib/engflow/worker/work/1/exec/bazel-out/k8-fastbuild/testlogs/pkg/ccl/testccl/workload/schemachange/schemachange_test/run_13_of_25/test.outputs/logTestWorkload4025396994
    panic.go:629: -- test log scope end --
test logs left over in: outputs.zip/logTestWorkload4025396994
--- FAIL: TestWorkload (10.95s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=13</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestWorkload.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

pkg/ccl/testccl/workload/schemachange/schemachange_test.TestWorkload [failed](https://tanzanite.cluster.engflow.com/invocations/default/f9978e6b-8d20-4dcd-aeaa-975a39247769?testReportRun=18&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL3Rlc3RjY2wvd29ya2xvYWQvc2NoZW1hY2hhbmdlOnNjaGVtYWNoYW5nZV90ZXN0) on master @ [472ea07a5232c98536293d13bb46cca59f9f2cd0](https://github.com/cockroachdb/cockroach/commits/472ea07a5232c98536293d13bb46cca59f9f2cd0):


```
    "query": "WITH tab_json AS (SELECT crdb_internal.pb_to_json('desc', descriptor)-\u003e'table' AS t FROM system.descriptor WHERE id = $1::REGCLASS), columns_json AS (SELECT json_array_elements(t-\u003e'columns') AS c FROM tab_json), columns AS (SELECT c-\u003e\u003e'computeExpr' AS generation_expression, c-\u003e\u003e'name' AS column_name, c-\u003e\u003e'id' AS ordinal FROM columns_json) SELECT generation_expression FROM columns WHERE generation_expression IS NOT NULL",
    "queryArgs": [
     "public.table_w0_3"
    ],
    "result": []
   }
  ],
  "previousStatements": [
   "ALTER TABLE public.table_w0_3 DROP COLUMN col_w0_3_w1_5"
  ],
  "usesDeclarativeSchemaChanger": true
 }
}
    schema_change_external_test.go:131: 
        	Error Trace:	pkg/ccl/testccl/workload/schemachange/schemachange_test_test/pkg/ccl/testccl/workload/schemachange/schema_change_external_test.go:131
        	Error:      	Received unexpected error:
        	            	***UNEXPECTED ERROR; Received an unexpected execution error.: ERROR: column "col_w0_3_w1_5" is referenced by the primary key (SQLSTATE 42P10)
        	            	(1) forced error mark
        	            	  | "fatal error when running txn"
        	            	  | github.com/cockroachdb/errors/withstack/*withstack.withStack::
        	            	Wraps: (2)
        	            	Wraps: (3) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*opStmt).executeStmt
        	            	  | 	github.com/cockroachdb/cockroach/pkg/workload/schemachange/operation_generator.go:3189
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*schemaChangeWorker).runInTxn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/workload/schemachange/schemachange.go:503
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*schemaChangeWorker).run
        	            	  | 	github.com/cockroachdb/cockroach/pkg/workload/schemachange/schemachange.go:600
        	            	  | pkg/ccl/testccl/workload/schemachange/schemachange_test_test.TestWorkload.TestWorkload.func4.func6
        	            	  | 	pkg/ccl/testccl/workload/schemachange/schemachange_test_test/pkg/ccl/testccl/workload/schemachange/schema_change_external_test.go:120
        	            	  | golang.org/x/sync/errgroup.(*Group).Go.func1
        	            	  | 	golang.org/x/sync/errgroup/external/org_golang_x_sync/errgroup/errgroup.go:78
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1695
        	            	Wraps: (4) ***UNEXPECTED ERROR; Received an unexpected execution error.
        	            	Wraps: (5) ERROR: column "col_w0_3_w1_5" is referenced by the primary key (SQLSTATE 42P10)
        	            	Error types: (1) *markers.withMark (2) *schemachange.ErrorState (3) *withstack.withStack (4) *errutil.withPrefix (5) *pgconn.PgError
        	Test:       	TestWorkload
Schema Workload Stats
Total Schema Statements Executed = 46
Total Schema Statements Succeeded = 46
Total Schema Statement Expected Failures = 0
Total Transactions Committed = 36
Total Transactions Rolled Back = 32
Total Transactions Executed = 68
    schema_change_external_test.go:96: backup, tracing data, and system table dumps in /var/lib/engflow/worker/work/1/exec/bazel-out/k8-fastbuild/testlogs/pkg/ccl/testccl/workload/schemachange/schemachange_test/run_18_of_25/test.outputs/logTestWorkload1257607946
    panic.go:626: -- test log scope end --
test logs left over in: outputs.zip/logTestWorkload1257607946
--- FAIL: TestWorkload (13.44s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=18</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #132904 pkg/ccl/testccl/workload/schemachange/schemachange_test: TestWorkload failed [C-test-failure O-robot T-sql-foundations branch-release-24.1 release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestWorkload.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

pkg/ccl/testccl/workload/schemachange/schemachange_test.TestWorkload [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/17605454?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/17605454?buildTab=artifacts#/) on master @ [5c74d5c52339e90615af026c8170b0deacd3b0a3](https://github.com/cockroachdb/cockroach/commits/5c74d5c52339e90615af026c8170b0deacd3b0a3):


```
    "query": "SELECT ARRAY[parent.table_schema, parent.table_name, parent.column_name, child.column_name] FROM (SELECT conname, conkey, confkey, conrelid, confrelid FROM pg_constraint WHERE (contype = 'f') AND (conrelid = 'public.table_w0_43'::REGCLASS::INT8)) AS con JOIN (SELECT column_name, ordinal_position, column_default FROM information_schema.columns WHERE (table_schema = 'public') AND (table_name = 'table_w0_43')) AS child ON conkey[1] = child.ordinal_position JOIN (SELECT pc.oid, cols.table_schema, cols.table_name, cols.column_name, cols.ordinal_position FROM pg_class AS pc JOIN pg_namespace AS pn ON pc.relnamespace = pn.oid JOIN information_schema.columns AS cols ON ((pc.relname = cols.table_name) AND (pn.nspname = cols.table_schema))) AS parent ON ((con.confkey[1] = parent.ordinal_position) AND (con.confrelid = parent.oid)) WHERE child.column_name != 'rowid'",
    "result": null
   }
  ],
  "previousStatements": [
   "INSERT INTO public.table_w0_43 (col43_w0_44, col43_w0_45, col43_w0_46, col43_w0_47, col43_w0_48, col43_w0_49, col43_w0_50, col43_w0_51, col43_w0_52) VALUES (2950:::OID,(-121):::INT8,'[-3.7093213,-9.434799e-15,-3.031264e+11,-1.609787e-35,-1.1490946e+21,8.093889e-22,-8.7726625e+21,-1.2168932e+17,4.571279e+21,-5.619166e+06,-1.5068127e-11,90902.57,-0.00013373098,1.1198205e-38,4.7402248e-23,-5.661567e-23,2.7454996e+38,2.0713318e-13,-1.8949986e-38,3.9702528e+36,0.099521354,-1.9664013e-19,2.6976579e-09,-3.8930374e-23,2.2679326e-26,-2.7886197e+28,69127.74,1.8110239e+10,-1.1990877e-06,5.2142127e-28,-1.0693503e+12,1.0681629e+06,4.8616187e+26,-7.440589e+36,0.00043641645,4.5522206e-12,-1.281008e-12,4.5991127e+29,-1.00667045e+29,-6.3353207e+37,-9.1756484e+27,-1.0521162e-36,-2.8433305e-13,-1.0798082e-09,-6.7543505e-38,3.6463673e-30,-1.8626858e-34,-1.906829e+36,-6.050032e-05,9.659988e+28,-0.00016234476,1.7645819e-24,-1.2389009e+29,7.731905e-09,0.37911844,7.833055e+16,-4.5944523e-07,9.8696704e-23,-1.766927e-36,3.1517097e+14,6.5155854e-15,2.975969e-33,-3.5790977e-20,-7.590962e+08,-2.2452821e-07,1.0625638e-15,-7.4062033,362236.1,-1.3153027e+31,-1.6920871e-24,1481.7786,-95.931015,1.8033767e+16,0.00034195845,-1.9940191e+34,-5.266941e-30,-2.360957e+34,5.9817693e+18,9.057104e+31,2.2903982e-23,2.2256638e-26,-1.8781985e-24,-1.134121e+14,-5.0332148e-37,7.72982e-18,-720.91,4.712682e+06,1.8928727e-29,4.398116e+22,827017.94,1.2671506e+13,-1.6290074e+30,-9.276937e-16,-3.8756095e+13,8.2775596e+35,-8.692296e-12,-5.224738e-15,-1.490225e+06,5.1652433e-17,145101.5,-23791.768,-6.7374845e-22,-1.3984871,-71707.37,1.5951324e-38,0.00057469145,79365.19,2.1504163e-19,-2.1172902e+28,9.018558e+08,-1.6905624e+22,-2.2004642e-38,2.5436372e-36,-3.1492249e-18,3.9552765e-16,4.361214e-31,4.6785924e+22,1.4900323e-16,1.1325327e-21,-1.1122623e-25,4.991981e+26,432847.25,-3.3233204e+26,-3.951335e+24,-1.8398586e+36,-6.008882e-08,224135.62,1.1959173e+22,-1.9531411e-24,3.32361e-25,-9.967856e+12,1.8092583e-23,-1.9547122e-10,-8.170283e+30,2.319489e-24,-2.7908534e+31,-2.9573914e-27,6.428423e-18,-114378.516,-1.3773945e+26,-5.046637e-10,-5.9920897e-18,-1.0488399e+24,-2.1705906e+29,5.5371515e+13,4.54506e-24,2.2127122e-37,-1.4964521e-10,6.714208e+28,6.530744e-07,2.094761e-29,-3.563009e-07,1.3132256e-08,1.4081498e+29,-7.6525906e-19,-4.6088938e+20,1.2190847e+10,1.4944053e-31,1.8136426e-08,-3255.6729,-3.6886615e-16,-2.6101477e+10,3.3055092e-28,1.7704526e+16,4.5745373e+10,-3.94433e+06,-217.96211,4.972059e+22,-1.441988e-23,3.195876e-07,-3.175127e-24,-1.9986678e+36,1.996553e-30,-3.4354876e+10,1.7554178e+19,1.3852727e-08,-1.1171976e+10,8.212075e-17,-1.0108444e-20,1.1651679e-25,7.9219646e+30,-1.7947416e+22,-1.2639081e+30,4.9013356e-17,4.4784053e+29,2016.4266,-4.272305e+33,-0.0005415709,-2.9746108e-35,1.939834e+24,-6.2112975e-30,-2.778473e+23,-3.8405493e-16,-5.055175e+33,1.27772376e-20,-4.5490186e+29,-5.772195e-10,1.9920128e+15,11.488941,2.0311537e-29,6.7711e+26,-1.6875295e-19,-4.8447402e+08,9.520574e+07,2.9964888e+24,-4.977267e-07,1.9661168e+17,-5.130345e-32,2.5423844e-10,-1.2426366e+11,2.266025e+14,4.145284e-09,9.338218e-23,-1.8830535e-18,-9.265094e-17,4.2113498e+09,-1.4087773e-10,6.5394705e-33,166763.75,-9.147548e+17,-3.5547396e+16,13116.602,-1.4413092e+11,8.988432e-16,7.7002944e+18,5.923899e+25,-3.340809e+26,-2.2196371e-16,9.064161e+32,-4.0198784e-27,7.970515e+29,-1.276327e+27,6.3976728e-12,-6.512721e+37,-4.8585164e+07,-3.9958488e-32,1.7804897e+17,-2.7047857e-33,1.8998703e-15,9.207821e+12,1.6673355e-33,-1.0461822e-27,7.456299e+22,3.7975842e-22,-9.707687e+07,5.1981948e+22,-2.2043428e+17,-1.05155455e+12,-7.151287e+13,1.3317603e+14,-1.8096146e-24,-3.9318008e+28,-4.5821234e-34,2.7680708e-32,2.1378168e-31,-9.331474e-14,-1.5862333e-22,31399.701,1.4320833e+34,-11.618629,1.4482812e+09,1.6509504e-35,-1.567378e-20,1.7803611e-12,-8.933342e-27,-1.1260947e-24,1.9720947e-15,-7.2468537e-10,3.3953346e+15,-4.0723118e+06,9.386059e+15,-1.7749917e+29,-3.3531898e+07,0.29035398,-4.3074462e+35,-6.0987724e+10,-3.4797098e+20,1.12916026e-35,1.7856018e-19,1.7000934e-14,0.0016449133,-0.00047754566,6.461123e+09,9.5339445e-11,-3.9316245e+23,-1.2219607e+16,15.02387,-7.0347605e-06,167602.66,8.060215e+28,-6.1176486e+32,-8.8683314e-38,5.7842794e+15,1.6806897e+13,4.0580646e+24,2.0940744e-10,170.4445,-0.002913867,1.7088907e-16,3.1948614e+07,7.980318e-35,-0.11099975,-1.7555358e+27,-0.0135302935,30411.057,2.104974e+25,-2.07887e+36,1.528812e-31,-2.3725692e-28,-1.1986763e-10,-3.2869698e-26,4.3283685e+30,6687.5054,7.064501e+25,-2.7667693e+15,-3.2359158e-35,2.8026179e-24,1.5473583e-34,-1.0061511e-29,2.4647896e-17,14.60031,-1.5790818e+25,1.523233e-22,-5.670043e+11,2.5989334e+25,3.2957576e+13,-1.8571066e+33,-1.53499e-16,-1.0104465e+20,4.6338868e+13,-1.6514817e-22,3.3487876e+25,-1.7627867e+27,2.5229867e+11,2.075481e-27,-2.016522e-25,-2.5536525,3.9791783e+21,0.24174301,7.7044776e+37,-4.469998e+07,-1.9379386e+31,6.006252e-39,-0.00042272964,-2.8265249e+23,-27.070526,-4.2152278e+29,-1.0590823e-34,8.7624185e+24,2.6188908e-33,4.7476404e+26,-9.927192e+36,-2.0485048e-25,-7.283212e+33,-1.6644833e+25,9.419189e+35,-9.620781e+09,1.5701511e-28,-8.8130115e-38,2.2380612e-36,-3.010944,-7.904978e-09,1.7855214e+38,-1.0276975e-11,-2.3231943e-32,-8.943132e+26,-5.7148313e+19,-0.042221136,3.075604e-15,9.197754e-05,9.592815e+34,-11.119661,-1.6082968e+22,-1.980343e-18,1.7968702e-31,-7.6166427e+31,1.9783314e-15,8.709913e-14,-1.3016655e+06,-6.796721e-15,-2.3423903e-26,-1.1313759e+19,14772.541,3.971407e-10,0.10745704,0.24379975,450685.2,3142.5427,-1.9343416e-16,1.710773e-28,0.32659757,-1.3301511e-05,5.64555e-18,-5.7630563e-22,5.8699083e-12,-6.005198e-09,1.1142796e-12,8.0653136e+20,6.9902527e-22,4.1986595e+16,-6.221388e+36,-117.996605,3.696964,-1.5075931e-37,-5.826863e-06,-1.9766858e-16,9.1283527e-26,-2.6174232e-34,-7.20243e-35,2.695369e+13,-4.364128e+29,-8.51141e-12,30060.018,-2.5927599e+16,-2.7300189e-30,1.5231777e+20,1.7787315e+36,3.241768e-07,4.5269222e+27,-1.56044e+36,5.4840316e-28,-0.114719786,-8.831101e-11,-1.4639538e-28,-2.1598543e-36,-1.9644177e+34,-665607.44,-1.4391248e+29,4.0904012e-32,-9.0933904e-17,8.0049e+23,3.5935784e+18,-7.190076e+15,-5.341002e-37,0.6624395,-1.3008572e+08,40.934002,-3.3511055e+10,6.5094116e-37,-0.00011880978,-3.199951,1.6455472e+12,-4.1383526e+08,1.0592797e-30,-2.7493002e+14,-1.9068248e+26,-7.851951e+06,3.9000135e+37,9.978619e+30,6.6521196e+13,2.791763e-39,-4.030774e+29,-2.3833013e+31,1.4496471e-28,5.63804e+29,1.3707496e+14,5.113566e+21,1.1494208e-35,1.13981666e+33,-6.3739437e+08,-3.3063894e+29,4.1682946e+32,4.8630918e+14,2.704847e+35,-8.3769335e-37,5.0806147e+25,-3.5548948e+27,345.75372,-7.983586e-29,8.996702e+31,-5.731247e-25,-1.1437523e-38,-3.546693e-08,485172.5,-2.3851705e-25,3.443406e+26,-4.705861e-20,3.4657452e+28,-2.2551398e+10,0.027182166,-3.9759607e+15,2.7181316e+17,0.00033629825,-9.828859e+34,9.771448e+36,-5.5630396e-24,-1.1722862e-14,-0.005509604,1.05279656e+17,-4.6782025e+31,-1217.7731,1.4826942e-36,1.1901882e-24,8.508562e-13,-8.1803215e+14,1592.3982,3.1883198e+07,84959.91,-5.465422e-18,2.4332075e-12,-8472.571,0.009059149,-0.941764,-4.5674554e+08,-8.653171e-36,-5.115298e+07,3.108834e-36,1.0300152e+22,6.7345716e-09,-0.013917588,-8.9643295e+21,-2.6913071e-14,3.237779e+14,-2.6659806e+07,-1.7417954e-28,-2.5316512e-34,-1.277178e+33,6.661278e-10,-5.058595e+14,6.505523e+13,6.0279266e+13,-7995.609,2.4069364e-33,-0.0023470887,3.819723e-30,5.3450444e+13,0.00020684877,-4.5928792e+11,-3.5487908e+11,9.0581736e-05,4.689463e+22,-1.7348369e+13,1.0837135e-18,5.405345e-17,2.6931872e-28,4.310367e-19,7.5856946e-16,4.925518e-06,4.6944687e+34,6.283898e-10,1.1134557e+32,-3.449966e-18,1.1779021e+30,1.1015913e-33,1.7521343e-37,3.675463e-28,1.1043754e+10,-1.9340041e-32,-3.5608802e+23,164.84647,-6.0653774e-06,-3.9890189e-13,-5.320105e-12,1.2563422e-24,2.9334238e-22,84645.66,-1.7860374e-12,-3.3058158e-38,-6.9712844e+17,-3.672836e-21,-5.946147e-32,-4.3345543e-28,-902.7319,-3.9243524e-18,-6.744444e+08,-4.774171e-30,2.870745,1.04053e+21,-1.134464e+27,-5.385484e+14,-2.0501991e+10,-2.4033598e+08,7.1158246e-07,2.5571501e-14,-8.958213e-24,-7.24257e+29,4.5353956e+11,9.7871495e-24,0.00026400445,7.7424676e+36,-8.769963e-21,-0.00047383964,5.0391933e+19,1.4958147e-08,-6.767068e-19,8.10053e+19,4.070522e+37,-1.5806551e+10,72659.62,0.00024058652,-9.609486e-37,2.2204918e+06,1.8899824e+19,-2.8399252e+07,-6.463554e-21,-7283.235,-1.0458409e-12,5.1789123e-10,-3.2021358,236.91087,-4.211229e+29,7.579528e-17,-2.933001e-29,1.230831e-06,-9.3032465e-12,-3.8665395e+35,-1.0483109e+23,5.453279e-22,-4.2447584e-18,3.1079785e-18,1.3511633e+16,2.9527006e-10,-0.15633358,-2.2503508e-11,5.259462e-38,-5.3287492e-20,-52.625465,2.7632263e-32,1.0141659e-18,-3.4470955e+24,9.070159e-08,-1.4188427e+27,5.6514637e+34,-3.4602376e+12,-2.0123333e-24,6.190379e-23,4.8803398e-26,-3.5412727e+22,-1.01141756e+06,6.7800353e+12,5.7206347e+32,-692.20593,2.7413191e+11,-1.857721e+33,-1.8895714e+33,-8.918295e+24,3.537871e-23,-3.099456e+25,-6.266635e+11,-5.565494e-14,1.3784558e+25,-0.9396922,-2.138355e+11,3.419692e-16,-8.184455e-06,-1.151393e-16,-2.2029569e-14,5.1508916e-05,4.039635e+10,-1.8702342e-17,2.4089547e+08,1.5525256e+09,116151.47,1.867048e-11,3.1503602e+29,-2.3798744e-34,-3.5160666e-19,1.5036195e+16,2.8350688e-12,-94.76575,5.8791212e-30,6.128689e+06,-2.8694326e-16,-3.2446502e+07,-3.04865e-40,-4.090169e+19,-1.2038277e-33,-8.658888e+17,-2.147148e-17,-1.1623577e+29,3.3623892e+16,-7.797196e-36,2257.0398,1.5434435e-28,-1.0893745e+19,2.6598035e-30,2.1262762e-25,2.7778026e+38,-2.1806113e-07,5.642096e+22,-31135.75,1.1217507e+17,7.9840296e+30,5.0700763e+22,-1.2941206e+22,-1.9680349e-20,-3.6714177e+34,-2.978516e-05,4.1258235e-32,3.9512137e+33,1.3606004e-34,1.4728513e-36,-3.2629652e-37,3.3650388e-13,-1.0700803e+09,775.68726,-1.7128548e-21,-3.3673555e-08,2.0981777e+28,-5.843855e-32,-7.396318e-16,-2.2739012e-19,5.479153e-20,-1.4660779e+38,-1.334599e-26,104.71528,2.7796706e-32,8.143305e+06,-4.3459475e+15,-9.714474e-11,-2.040822e-18,-9.92126e-09,-3.833058e-30,-1.1846501e+17,-8.26459e-31,-5.239673e+29,-3.6868922e-31,-2.5316386e+21,-2.167275e-24,1.2450405e+31,-1.3578286e-25,2.7164043e+17,0.014267497,-8.0257926e+36,3.1824363e+15,3.661671e+11,4.512117e-25,4.3481766e+08,-1.8417782e+16,2.0670227e+23,1.5893851e-34,5.9461345e+06,1.387766e+30,7.4294676e-14,1.1194525e+33,-1.6131104e-37,-1.3368781e+20,-1.6623028e+14,-8.5483734e-08,0.014485991,-1.096474e-37,9.841788e-10,3.37273e+13,7.385138e-10,-7.3341004e-35,-26.982042,-3.386891e-05,1.2208958e+30,-2.609455e-12,3.582304e-30,9.567564e+26,-1.8538519e+28,1.7695924e-12,-1.9994466e+27,0.0007060024,-7.46583e-33,9.0209206e-32,2.3895092e+24,1.2171428e-15]':::VECTOR,(-38):::INT8,ARRAY[(-27064):::INT8,(-7610):::INT8,(-31087):::INT8,(-6858):::INT8],'7D019C16/C3B67EAE':::PG_LSN,1560:::OID,0.7672098699637303:::FLOAT8,e'\\x1fu'::STRING),(1083:::OID,(-58):::INT8,'[1.1427341e+31,-2.074233e-20,3.4166756e-11,-1.6898627e-31,4.854765e-39,-1.4853246e+11,213.33453,3.8371254e+19,1.1777461e-07,5.7623424e-12,-3.4763257e+30,2.88268e-10,2.172964e-06,-3.7978646e+30,-6.604368,-1.0411929e+31,-6.466354e-33,-7.565854e+30,5.862871e+20,1.7738467e-23,-2.7154865e+11,-3.969274e+19,57036.54,-1.792661e-33,2.245034e-09,5.6612285e-34,5.670259e+14,4.486948e-27,-0.0012570892,-0.004967086,0.24581324,7.0470685e-11,7.54822e+33,1.3694421e-07,8.7769724e+15,-7.545765e+31,-9.485464e-33,7.009495e-09,1.4996092e-12,7.85556e+13,2.8588014e+28,-1.4549048e+23,-1.9750275e-20,2.2357744e+15,1.5300348e+30,-4.9404304e-29,-9.6009935e-06,-4.6039585e+06,-3.9012185e-29,-6.2255513e+24,-1.5974636e-33,-5.5459174e+17,-1.1093896e+30,-6.4609054e+12,-5.007183e+06,-46.011494,3.2024954e-19,5.5138903e-23,-1.7753304e-17,1.8272295e+22,-3.1171722e+14,-765.1997,5.5237574e+14,7.586471e+25,-4.0876146e-19,3.9839366e+37,1.9882692e+16,-1.682081e-29,-2.369669e-21,-19041.508,4.2989056e-17,3.275313e-25,4.0533273e+31,2.6199316e+20,5.925099e-15,3.8116265e+35,8.244457e+14,-4.0189622e+34,1.8163584e+24,5.576281e+10,-8.6107686e+08,-2.7645958e-14,-5.5820136e+22,2.4727197e+21,-3.4418152e-20,2.3931779e+36,1.676743e+17,4.9346727e+28,5.1355796e+13,-1.9024254e+30,1.7114117e+29,3.3312927e+37,1.0814937e-26,165.69514,-4.2762442e-36,1.7958428e+35,-26.673481,-8.467562e+06,4.4695426e-29,-4.3664107e+21,-3.5391764e-08,3.959061e-10,-1.0719124e-36,-8.812685e+22,-1.5590208e+19,0.0022129368,2.4070045e-24,5.0620164e+12,1293.4127,3.6295913e+26,8.2350953e-19,5.384012e+37,1.10265096e-10,2.3741254e+10,1.0034906e+15,2.8937398e-08,9.956631e+29,-3.2083062e-29,-8.539607e+24,2.8593545e+21,1.5450703e-38,-1.1995553e+16,-1.2787315e+18,-8.703283e-25,3.8023213e-31,5.948238e-24,-1.617705e-13,-2.8430844e-07,-6.93186e-28,1.1991796e-20,-2.1173812e-27,1.3745347e+33,482209.72,4.749301,-8.743032e+20,1.2660817e+28,2.1292555e-15,2.5057743e+34,3303.4766,-1.0570091e+07,-4307.1055,-1.1834856e+10,1.3079909e-34,4.1707446e-35,2.7803944e-21,2.3633917e+28,2.1374534e-08,1.6031003e+15,2.3970348e-33,5.0288528e+17,-2.9292723e+08,2.4741243e-23,-494857,-790103.44,-6.105599e-31,1.3417215e+36,-62862.137,-1.943099e-06,8.3898564e-38,-5.3266785e+14,4.228833e+15,9.126287e-10,1.6589821e+37,2.346696e+29,-9.257806e-26,-866.4242,9.8673025e-20,4.246026e+20,1.2891513e-36,-4.0439793e-19,-4.022041e-19,1.1549558e-05,9.248183e-09,1.1956719e+32,-8.1278966e-18,1.1132622e+14,7.586634e-06,-1.5734298e-28,-8.954715e+12,6.654725e-32,-1.4822453e+07,1.8670075e-10,-2.7948776e+13,1.1415831e-14,0.065771095,-2.8918626e-36,-7.123083e+12,5.2614496e-27,1.3405789e+19,-6.1881287e-24,-2.4719162e-35,2.925181e-36,84.23209,-4.624341e-10,-2.171573e+33,-12.265624,1.369686e-36,9.950422e-31,-7.2391195e-09,-1.7372747e-12,9.111441e-12,5.3141263e-15,-2.0476963e+33,7.1028314e+18,0.0009424147,-5.5994097e-28,7.8241794e+30,-1.4141197e+18,-1.483911e-21,2.5273555e-32,3.2404495e-26,5.962325e-18,8.6306656e-08,-5.7106765e-16,-3.5318833e+27,-5.7145675e-15,3.2375536e-19,1.5702881e-13,-2.3531994e-21,-0.04360991,6.6590294e+29,1.9931173e-08,8.1415845e+11,-2.1880302e+36,1.6589182e+31,-1.6919832e+07,-6.992457e+29,4.585757e-34,-49.749527,-3.8554869,2.2102674e+30,2.1544266e+21,1.1938789e+15,-1.4049895e+17,3.9982733e-32,1.1686868e+38,-0.0019283912,1.1194133e-14,-1.4859051e-33,-2.533468e+25,2.0241234e-11,3.4327216e-25,8.2962626e-38,-1.376793e-07,-4.929908e+12,-3.331633e+08,1.645658e+10,3.5864815e-22,4.81099e-12,-1.6374532e-09,-3.0652141e-18,1.5750175e-21,-0.16611154,-1.6691523e+15,-5.948837e+12,6.4819696e+07,-4.656034e+18,0.019233918,-4.386101e+23,-1.6461359e-18,6.643592e+28,-1.0349662e-22,-4.5938803e+18,1.0371715e-34,1.0841291e+35,1.668159e-08,-2.6108019e-11,-6.8235373e-09,315.2599,-5.784997e+35,-1.1712122e-35,-5.715942e+18,0.15833376,-1.6294501e-21,5.916728e+17,0.036020514,-8.396442e-13,-8.417522e-20,-1.8760889e-23,1.6866192e-11,-8.08169e-14,-2.5945987e-32,-5.3141006e+14,-1.9645005e+30,5.258764e+08,-4.301925e-07,-1.8553576e-05,-1.6093783e-30,1.1900403e-32,-7.32643e-31,-2.2734018e+30,6.331459e-25,1.7973985e-12,-6.5444055e-30,1.5814333e-12,8.894579e+17,-8.074477e+08,-7.540342e-27,-6.993835e+11,-7.66046e-34,6.837315,1.8377792e-32,3.7067068e-08,-1.11278935e+17,1.4783419e+23,1.78912e-25,-9.899167e+14,4.270505e-08,-3.1797482e-33,-6.4274726e+33,-2.2260348e+24,-1.1408997e+33,-2.7286118e+17,-2.0270223e+09,1.6604927e-09,-138407.52,-5.0483107e+23,1.2871973e+37,-1.8250358e+12,1.4656083e+07,-3.8888546e+33,-1.4351256e+19,1.6736261e-25,-727.64825,-3.1265346e-15,6.311546e-37,2.9827691e-15,-9.405661e+32,0.0100857895,-160791.44,8.231454e+32,-2.2747413e+34,-9.3052645e+25,-3.955752e-30,-5.153089e+21,-4.241835e+10,-6.316338e+30,-1.1617253e+37,2.8002245e-38,-4.0606474e-16,-537599.8,2.6108648e-06,5.6502366e+23,-1.3334775e+38,128026.62,1.5130955e+26,3.2354555e-09,4.5849876e-37,-3.1029384e-16,2.4961367e+27,9.799355e-26,8.3966637e+12,2.8465448e+23,2.5131254,0.02115368,1.9383502e-34,-1.2247267e+28,6.013124e-08,-3.3038317e+34,-1.7569996,-1.3552151e+38,-0.12521736,1.864368e+33,2.6247438e-10,3.8459566e+17,5.34441e+06,-9.199602e-30,5.897557e-39,0.00019165072,6.396106e+36,-3.692121e-17,-2.539766e-37,-1.6125883e-21,3.381929e-20,1.6522665e+17,-3.2138626e+15,-1.9183623e+11,-3.6164414e-07,3.2891194e+33,3.4180632e-31,-9.388486e-38,-3.136248e+36,-2.9510387e+38,-2.466025e-24,-3.5410485e-28,-4.651168e-06,-0.003182993,-1.0264029e+29,-3.4777565e-07,2.6300234e-38,-17900.078,-1.7835174e-35,9.634245e-31,4.65343e+14,121.78019,-2.6729934e+37,0.07269695,0.00077384274,-5.7374493e+25,0.99743694,6.396492e-24,3.5266567e+34,-1.087712e-27,-3.2896787e+26,-3.5673769,3.830058e-36,-233112.66,1.0321966e+14,5.5586056e-06,-1.1758271e+31,-9.251742e+07,3.84401e+22,-8.25811e-28,5.974414e-28,6.756243e-15,1.284283e-35,-2.974926e-18,4.284097e-20,-1.2262163e+11,-1.3019926e+18,-1.5910452e+32,-3.047868e+24,2.0458661e-10,-2.5647226e-11,-770650.1,1.0394021e+14,-1.3322994e+10,0.052537717,2.2774799e+35,3.9118368e+12,-1.4211539e+09,1.1455985e+10,1.1938634e-23,-7.8444397e-25,9.757992e+15,9.335878e+24,8.1724784e+27,-1.918975e-33,5.715888e-19,-4.4664237e+13,-23020.318,5.425689e+14,-2.2350915e+17,-3.227213e+16,-8.451924e-12,-5.4642514e+31,-3.206242e+35,-2.6836657e-36,-3.704665e+19,-5.063253e+12,1.2337284e+12,-4.3615287e+34,-8.629729e-20,1.0516981e+18,-643896.4,-1.424149e-24,-1.3788863e-09,-9.101604e-25,5.174707e-18,-1.08835365e-13,0.29063118,1.3474213e+11,-2.9642037e-36,4.9431606e-18,9.4531075e-05,-2.5838451e+12]':::VECTOR,48:::INT8,ARRAY[(-24461):::INT8,(-80):::INT8],'BEE662F3/B4E0E38C':::PG_LSN,4096:::OID,(-0.7830855390230527):::FLOAT8,e'k?\\x01,tb'::STRING)"
  ],
  "usesDeclarativeSchemaChanger": true
 }
}
    schema_change_external_test.go:131: 
        	Error Trace:	pkg/ccl/testccl/workload/schemachange/schema_change_external_test.go:131
        	Error:      	Received unexpected error:
        	            	***UNEXPECTED ERROR; Received an unexpected execution error.: ERROR: command is too large: 93103751 bytes (max: 67108864) (SQLSTATE XXUUU)
        	            	(1) forced error mark
        	            	  | "txn needs to rollback"
        	            	  | github.com/cockroachdb/errors/withstack/*withstack.withStack::
        	            	Wraps: (2) forced error mark
        	            	  | "fatal error when running txn"
        	            	  | github.com/cockroachdb/errors/withstack/*withstack.withStack::
        	            	Wraps: (3)
        	            	Wraps: (4) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*opStmt).executeStmt
        	            	  | 	pkg/workload/schemachange/operation_generator.go:3220
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*schemaChangeWorker).runInTxn
        	            	  | 	pkg/workload/schemachange/schemachange.go:503
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*schemaChangeWorker).run
        	            	  | 	pkg/workload/schemachange/schemachange.go:600
        	            	  | pkg/ccl/testccl/workload/schemachange/schemachange_test_test.TestWorkload.TestWorkload.func4.func6
        	            	  | 	pkg/ccl/testccl/workload/schemachange/schema_change_external_test.go:120
        	            	  | golang.org/x/sync/errgroup.(*Group).Go.func1
        	            	  | 	external/org_golang_x_sync/errgroup/errgroup.go:78
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_arm64.s:1222
        	            	Wraps: (5) ***UNEXPECTED ERROR; Received an unexpected execution error.
        	            	Wraps: (6) ERROR: command is too large: 93103751 bytes (max: 67108864) (SQLSTATE XXUUU)
        	            	Error types: (1) *markers.withMark (2) *markers.withMark (3) *schemachange.ErrorState (4) *withstack.withStack (5) *errutil.withPrefix (6) *pgconn.PgError
        	Test:       	TestWorkload
Schema Workload Stats
Total Schema Statements Executed = 19
Total Schema Statements Succeeded = 19
Total Schema Statement Expected Failures = 0
Total Transactions Committed = 11
Total Transactions Rolled Back = 24
Total Transactions Executed = 35
    schema_change_external_test.go:96: backup, tracing data, and system table dumps in /artifacts/tmp/_tmp/67537770b84c3a4ea177539c1e46bd43/logTestWorkload3405335784
    panic.go:626: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/67537770b84c3a4ea177539c1e46bd43/logTestWorkload3405335784
--- FAIL: TestWorkload (5.81s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #134056 pkg/ccl/testccl/workload/schemachange/schemachange_test: TestWorkload failed [C-test-failure O-robot T-sql-foundations branch-release-24.3 release-blocker]
- #134046 pkg/ccl/testccl/workload/schemachange/schemachange_test: TestWorkload failed [C-test-failure O-robot T-sql-foundations branch-release-23.1 release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestWorkload.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/ccl/testccl/workload/schemachange/schemachange_test.TestWorkload [failed](https://tanzanite.cluster.engflow.com/invocations/default/7a043f45-0327-499c-88c8-5a3f68932ec2?testReportRun=17&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL3Rlc3RjY2wvd29ya2xvYWQvc2NoZW1hY2hhbmdlOnNjaGVtYWNoYW5nZV90ZXN0) on master @ [688e82e8d015350fe3aa263484416d28b232a25d](https://github.com/cockroachdb/cockroach/commits/688e82e8d015350fe3aa263484416d28b232a25d):


```
    "result": [
     "lower(col5_w0_10)",
     "lower(col5_w0_10)",
     "lower(col5_w0_6)"
    ]
   }
  ],
  "previousStatements": [
   "ALTER TABLE public.table_w0_32 DROP COLUMN col5_w0_13"
  ],
  "usesDeclarativeSchemaChanger": true
 }
}
    schema_change_external_test.go:131: 
        	Error Trace:	pkg/ccl/testccl/workload/schemachange/schema_change_external_test.go:131
        	Error:      	Received unexpected error:
        	            	***UNEXPECTED ERROR; Received an unexpected execution error.: ERROR: column "col5_w0_13" is referenced by the primary key (SQLSTATE 42P10)
        	            	(1) forced error mark
        	            	  | "fatal error when running txn"
        	            	  | github.com/cockroachdb/errors/withstack/*withstack.withStack::
        	            	Wraps: (2)
        	            	Wraps: (3) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*opStmt).executeStmt
        	            	  | 	pkg/workload/schemachange/operation_generator.go:3233
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*schemaChangeWorker).runInTxn
        	            	  | 	pkg/workload/schemachange/schemachange.go:503
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*schemaChangeWorker).run
        	            	  | 	pkg/workload/schemachange/schemachange.go:600
        	            	  | pkg/ccl/testccl/workload/schemachange/schemachange_test_test.TestWorkload.TestWorkload.func4.func6
        	            	  | 	pkg/ccl/testccl/workload/schemachange/schema_change_external_test.go:120
        	            	  | golang.org/x/sync/errgroup.(*Group).Go.func1
        	            	  | 	external/org_golang_x_sync/errgroup/errgroup.go:78
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1695
        	            	Wraps: (4) ***UNEXPECTED ERROR; Received an unexpected execution error.
        	            	Wraps: (5) ERROR: column "col5_w0_13" is referenced by the primary key (SQLSTATE 42P10)
        	            	Error types: (1) *markers.withMark (2) *schemachange.ErrorState (3) *withstack.withStack (4) *errutil.withPrefix (5) *pgconn.PgError
        	Test:       	TestWorkload
Schema Workload Stats
Total Schema Statements Executed = 29
Total Schema Statements Succeeded = 29
Total Schema Statement Expected Failures = 0
Total Transactions Committed = 21
Total Transactions Rolled Back = 17
Total Transactions Executed = 38
    schema_change_external_test.go:96: backup, tracing data, and system table dumps in /var/lib/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/testlogs/pkg/ccl/testccl/workload/schemachange/schemachange_test/run_17_of_25/test.outputs/logTestWorkload2987934771
    panic.go:626: -- test log scope end --
test logs left over in: outputs.zip/logTestWorkload2987934771
--- FAIL: TestWorkload (10.83s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=17</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #134652 pkg/ccl/testccl/workload/schemachange/schemachange_test: TestWorkload failed [C-test-failure O-robot T-sql-foundations branch-release-24.1.7-rc release-blocker]
- #134401 pkg/ccl/testccl/workload/schemachange/schemachange_test: TestWorkload failed [C-test-failure O-robot T-sql-foundations branch-release-24.1 release-blocker]
- #134056 pkg/ccl/testccl/workload/schemachange/schemachange_test: TestWorkload failed [C-test-failure O-robot P-2 T-sql-foundations branch-release-24.3]
- #134046 pkg/ccl/testccl/workload/schemachange/schemachange_test: TestWorkload failed [C-test-failure O-robot P-2 T-sql-foundations branch-release-23.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestWorkload.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/ccl/testccl/workload/schemachange/schemachange_test.TestWorkload [failed](https://tanzanite.cluster.engflow.com/invocations/default/e21ae65a-761e-4b0f-9408-c3626522c91f?testReportRun=15&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL3Rlc3RjY2wvd29ya2xvYWQvc2NoZW1hY2hhbmdlOnNjaGVtYWNoYW5nZV90ZXN0) on master @ [6610d705724a21c836f3521f75972e65d9e9e2d4](https://github.com/cockroachdb/cockroach/commits/6610d705724a21c836f3521f75972e65d9e9e2d4):


```
    "result": false
   },
   {
    "query": "SELECT ARRAY[parent.table_schema, parent.table_name, parent.column_name, child.column_name] FROM (SELECT conname, conkey, confkey, conrelid, confrelid FROM pg_constraint WHERE (contype = 'f') AND (conrelid = 'schema_w1_70.table_w1_96'::REGCLASS::INT8)) AS con JOIN (SELECT column_name, ordinal_position, column_default FROM information_schema.columns WHERE (table_schema = 'schema_w1_70') AND (table_name = 'table_w1_96')) AS child ON conkey[1] = child.ordinal_position JOIN (SELECT pc.oid, cols.table_schema, cols.table_name, cols.column_name, cols.ordinal_position FROM pg_class AS pc JOIN pg_namespace AS pn ON pc.relnamespace = pn.oid JOIN information_schema.columns AS cols ON ((pc.relname = cols.table_name) AND (pn.nspname = cols.table_schema))) AS parent ON ((con.confkey[1] = parent.ordinal_position) AND (con.confrelid = parent.oid)) WHERE child.column_name != 'rowid'",
    "result": null
   }
  ],
  "previousStatements": [
   "INSERT INTO schema_w1_70.table_w1_96 (col96_w1_97, col96_w1_98, col96_w1_99, col96_w1_100, col96_w1_101, col96_w1_102, col96_w1_103, col96_w1_104, col96_w1_105, col96_w1_106, col96_w1_107, col96_w1_108) VALUES (NULL,'1995-07-01 14:10:30.000511+00':::TIMESTAMPTZ,25:::OID,ARRAY['1988-11-03 06:51:47.000497+00':::TIMESTAMPTZ,'1985-01-13 22:11:03.000216+00':::TIMESTAMPTZ,'4714-11-24 00:00:00+00 BC':::TIMESTAMPTZ,'1988-10-27 13:30:01.000374+00':::TIMESTAMPTZ,'1992-01-15 20:57:48.000498+00':::TIMESTAMPTZ],16:::OID,0.7055497488726438:::FLOAT8,false,'aebfcbaf-23f7-4752-912e-dcf9efb54d95':::UUID,'BOX(-0.46779095341686416 -0.7238280682750018,1.0528907020397904 0.48217627725666967)':::BOX2D,B'0011000','{\"7abAweO\": true, \"S10sA\": [\"a\", false, 0.43033379627815815, false]}':::JSONB,90002:::OID),('18:32:45.673281':::TIME,'1986-07-20 18:15:12.000193+00':::TIMESTAMPTZ,3614:::OID,ARRAY['2012-01-04 07:13:18.000519+00':::TIMESTAMPTZ,'2001-10-19 14:00:45.000986+00':::TIMESTAMPTZ,'4714-11-24 00:00:00+00 BC':::TIMESTAMPTZ,'1996-10-15 09:38:52.000754+00':::TIMESTAMPTZ,'2005-01-03 20:54:08.000515+00':::TIMESTAMPTZ],20:::OID,(-0.40758238303841715):::FLOAT8,true,'baa7788b-4fff-4392-86dc-a639844fb372':::UUID,'BOX(-2.07515538091374 -0.22374012898117385,-0.07375697099855405 1.0007040902866082)':::BOX2D,B'1100101','[{\"I2UQ\": \"b\", \"JZZQtx8X\": \"foo\", \"b\": false, \"sTyYSS7\": {\"NAWW\": false, \"a\": false}}, [null, [true], false, false], false]':::JSONB,90000:::OID),('00:00:00':::TIME,'1978-02-05 16:05:08.000224+00':::TIMESTAMPTZ,20:::OID,ARRAY['1998-07-02 04:35:44.000583+00':::TIMESTAMPTZ,'2011-05-01 22:42:20.000021+00':::TIMESTAMPTZ,'4714-11-24 00:00:00+00 BC':::TIMESTAMPTZ],23:::OID,(-2.4521872118052297):::FLOAT8,false,'7362ee8f-06b1-49b5-ae41-78134505f190':::UUID,'BOX(-0.6386927058871754 -1.0659066167933862,-0.037731470778560094 0.5065116104275563)':::BOX2D,B'1111100','{\"64y\": \"b\", \"a\": \"LnGxj9Ot\", \"bar\": \"baz\", \"h1uKHcOx\": {\"sCMNYHt\": true}}':::JSONB,1560:::OID)"
  ],
  "usesDeclarativeSchemaChanger": true
 }
}
    schema_change_external_test.go:131: 
        	Error Trace:	pkg/ccl/testccl/workload/schemachange/schema_change_external_test.go:131
        	Error:      	Received unexpected error:
        	            	***UNEXPECTED ERROR; Received an unexpected execution error.: ERROR: bit string length 0 does not match type BIT(18) (SQLSTATE 22026)
        	            	(1) forced error mark
        	            	  | "fatal error when running txn"
        	            	  | github.com/cockroachdb/errors/withstack/*withstack.withStack::
        	            	Wraps: (2)
        	            	Wraps: (3) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*opStmt).executeStmt
        	            	  | 	pkg/workload/schemachange/operation_generator.go:3228
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*schemaChangeWorker).runInTxn
        	            	  | 	pkg/workload/schemachange/schemachange.go:503
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*schemaChangeWorker).run
        	            	  | 	pkg/workload/schemachange/schemachange.go:600
        	            	  | pkg/ccl/testccl/workload/schemachange/schemachange_test_test.TestWorkload.TestWorkload.func4.func6
        	            	  | 	pkg/ccl/testccl/workload/schemachange/schema_change_external_test.go:120
        	            	  | golang.org/x/sync/errgroup.(*Group).Go.func1
        	            	  | 	external/org_golang_x_sync/errgroup/errgroup.go:78
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1695
        	            	Wraps: (4) ***UNEXPECTED ERROR; Received an unexpected execution error.
        	            	Wraps: (5) ERROR: bit string length 0 does not match type BIT(18) (SQLSTATE 22026)
        	            	Error types: (1) *markers.withMark (2) *schemachange.ErrorState (3) *withstack.withStack (4) *errutil.withPrefix (5) *pgconn.PgError
        	Test:       	TestWorkload
Schema Workload Stats
Total Schema Statements Executed = 46
Total Schema Statements Succeeded = 46
Total Schema Statement Expected Failures = 0
Total Transactions Committed = 29
Total Transactions Rolled Back = 29
Total Transactions Executed = 58
    schema_change_external_test.go:96: backup, tracing data, and system table dumps in /var/lib/engflow/worker/work/1/exec/bazel-out/k8-fastbuild/testlogs/pkg/ccl/testccl/workload/schemachange/schemachange_test/run_15_of_25/test.outputs/logTestWorkload1736456548
    panic.go:626: -- test log scope end --
test logs left over in: outputs.zip/logTestWorkload1736456548
--- FAIL: TestWorkload (9.34s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=15</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #135066 pkg/ccl/testccl/workload/schemachange/schemachange_test: TestWorkload failed [C-test-failure O-robot T-sql-foundations branch-release-24.3.0-rc]
- #134652 pkg/ccl/testccl/workload/schemachange/schemachange_test: TestWorkload failed [C-test-failure O-robot P-3 T-sql-foundations branch-release-24.1.7-rc]
- #134056 pkg/ccl/testccl/workload/schemachange/schemachange_test: TestWorkload failed [C-test-failure O-robot P-2 T-sql-foundations branch-release-24.3]
- #134046 pkg/ccl/testccl/workload/schemachange/schemachange_test: TestWorkload failed [C-test-failure O-robot P-2 T-sql-foundations branch-release-23.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestWorkload.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/ccl/testccl/workload/schemachange/schemachange_test.TestWorkload [failed](https://tanzanite.cluster.engflow.com/invocations/default/48f864ba-d888-45fe-aaaa-d95272da4799?testReportRun=9&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL3Rlc3RjY2wvd29ya2xvYWQvc2NoZW1hY2hhbmdlOnNjaGVtYWNoYW5nZV90ZXN0) on master @ [a5ab06046e817627cd0b64390f4ede0c4a5d82ef](https://github.com/cockroachdb/cockroach/commits/a5ab06046e817627cd0b64390f4ede0c4a5d82ef):


```
    "result": false
   },
   {
    "query": "SELECT ARRAY[parent.table_schema, parent.table_name, parent.column_name, child.column_name] FROM (SELECT conname, conkey, confkey, conrelid, confrelid FROM pg_constraint WHERE (contype = 'f') AND (conrelid = 'schema_w1_26.table_w1_28'::REGCLASS::INT8)) AS con JOIN (SELECT column_name, ordinal_position, column_default FROM information_schema.columns WHERE (table_schema = 'schema_w1_26') AND (table_name = 'table_w1_28')) AS child ON conkey[1] = child.ordinal_position JOIN (SELECT pc.oid, cols.table_schema, cols.table_name, cols.column_name, cols.ordinal_position FROM pg_class AS pc JOIN pg_namespace AS pn ON pc.relnamespace = pn.oid JOIN information_schema.columns AS cols ON ((pc.relname = cols.table_name) AND (pn.nspname = cols.table_schema))) AS parent ON ((con.confkey[1] = parent.ordinal_position) AND (con.confrelid = parent.oid)) WHERE child.column_name != 'rowid'",
    "result": null
   }
  ],
  "previousStatements": [
   "INSERT INTO schema_w1_26.table_w1_28 (col28_w1_29, col28_w1_30, col28_w1_31, col28_w1_32) VALUES (ARRAY[(-0.08362699176331954238):::DECIMAL,6.276775258894790156E+28:::DECIMAL,(-1):::DECIMAL,5.755137042381584177E+25:::DECIMAL],true,2205:::OID,(-1.185435687819133):::FLOAT8),(ARRAY[(-3.229614298631276472E+26):::DECIMAL],true,1790:::OID,2.6877142823633493:::FLOAT8),(ARRAY[(-1.28394730141230869E+27):::DECIMAL],true,1114:::OID,1.7976931348623157e+308:::FLOAT8)"
  ],
  "usesDeclarativeSchemaChanger": true
 }
}
    schema_change_external_test.go:131: 
        	Error Trace:	pkg/ccl/testccl/workload/schemachange/schema_change_external_test.go:131
        	Error:      	Received unexpected error:
        	            	***UNEXPECTED ERROR; Received an unexpected execution error.: ERROR: duplicate key value violates unique constraint "table_w1_28_col_w1_28_w0_64_key" (SQLSTATE 23505)
        	            	(1) forced error mark
        	            	  | "fatal error when running txn"
        	            	  | github.com/cockroachdb/errors/withstack/*withstack.withStack::
        	            	Wraps: (2)
        	            	Wraps: (3) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*opStmt).executeStmt
        	            	  | 	pkg/workload/schemachange/operation_generator.go:3228
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*schemaChangeWorker).runInTxn
        	            	  | 	pkg/workload/schemachange/schemachange.go:503
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*schemaChangeWorker).run
        	            	  | 	pkg/workload/schemachange/schemachange.go:600
        	            	  | pkg/ccl/testccl/workload/schemachange/schemachange_test_test.TestWorkload.TestWorkload.func4.func6
        	            	  | 	pkg/ccl/testccl/workload/schemachange/schema_change_external_test.go:120
        	            	  | golang.org/x/sync/errgroup.(*Group).Go.func1
        	            	  | 	external/org_golang_x_sync/errgroup/errgroup.go:78
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1695
        	            	Wraps: (4) ***UNEXPECTED ERROR; Received an unexpected execution error.
        	            	Wraps: (5) ERROR: duplicate key value violates unique constraint "table_w1_28_col_w1_28_w0_64_key" (SQLSTATE 23505)
        	            	Error types: (1) *markers.withMark (2) *schemachange.ErrorState (3) *withstack.withStack (4) *errutil.withPrefix (5) *pgconn.PgError
        	Test:       	TestWorkload
Schema Workload Stats
Total Schema Statements Executed = 49
Total Schema Statements Succeeded = 49
Total Schema Statement Expected Failures = 0
Total Transactions Committed = 35
Total Transactions Rolled Back = 32
Total Transactions Executed = 67
    schema_change_external_test.go:96: backup, tracing data, and system table dumps in /var/lib/engflow/worker/work/1/exec/bazel-out/k8-fastbuild/testlogs/pkg/ccl/testccl/workload/schemachange/schemachange_test/run_9_of_25/test.outputs/logTestWorkload1634885882
    panic.go:626: -- test log scope end --
test logs left over in: outputs.zip/logTestWorkload1634885882
--- FAIL: TestWorkload (15.44s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=9</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #135066 pkg/ccl/testccl/workload/schemachange/schemachange_test: TestWorkload failed [C-test-failure O-robot T-sql-foundations branch-release-24.3.0-rc]
- #134652 pkg/ccl/testccl/workload/schemachange/schemachange_test: TestWorkload failed [C-test-failure O-robot P-3 T-sql-foundations branch-release-24.1.7-rc]
- #134056 pkg/ccl/testccl/workload/schemachange/schemachange_test: TestWorkload failed [C-test-failure O-robot P-2 T-sql-foundations branch-release-24.3]
- #134046 pkg/ccl/testccl/workload/schemachange/schemachange_test: TestWorkload failed [C-test-failure O-robot P-2 T-sql-foundations branch-release-23.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestWorkload.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/ccl/testccl/workload/schemachange/schemachange_test.TestWorkload [failed](https://tanzanite.cluster.engflow.com/invocations/default/f6d49411-9187-47de-9d67-f9f1baa46894?testReportRun=16&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL3Rlc3RjY2wvd29ya2xvYWQvc2NoZW1hY2hhbmdlOnNjaGVtYWNoYW5nZV90ZXN0) on master @ [8cf1ed260d43755a694abe1b3bd76c599a0f9356](https://github.com/cockroachdb/cockroach/commits/8cf1ed260d43755a694abe1b3bd76c599a0f9356):


```
    ],
    "result": [
     "abs(col222_w0_225)",
     "CASE WHEN col222_w0_223 IS NULL THEN 'v3Lw]':::STRING ELSE e'z\\b\\x14q/}D':::STRING END"
    ]
   }
  ],
  "previousStatements": [
   "ALTER TABLE schema_w1_84.table_w0_222 DROP COLUMN col222_w0_226"
  ],
  "usesDeclarativeSchemaChanger": true
 }
}
    schema_change_external_test.go:131: 
        	Error Trace:	pkg/ccl/testccl/workload/schemachange/schema_change_external_test.go:131
        	Error:      	Received unexpected error:
        	            	***UNEXPECTED ERROR; Received an unexpected execution error.: ERROR: column "col222_w0_226" is referenced by the primary key (SQLSTATE 42P10)
        	            	(1) forced error mark
        	            	  | "fatal error when running txn"
        	            	  | github.com/cockroachdb/errors/withstack/*withstack.withStack::
        	            	Wraps: (2)
        	            	Wraps: (3) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*opStmt).executeStmt
        	            	  | 	pkg/workload/schemachange/operation_generator.go:3228
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*schemaChangeWorker).runInTxn
        	            	  | 	pkg/workload/schemachange/schemachange.go:503
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*schemaChangeWorker).run
        	            	  | 	pkg/workload/schemachange/schemachange.go:600
        	            	  | pkg/ccl/testccl/workload/schemachange/schemachange_test_test.TestWorkload.TestWorkload.func4.func6
        	            	  | 	pkg/ccl/testccl/workload/schemachange/schema_change_external_test.go:120
        	            	  | golang.org/x/sync/errgroup.(*Group).Go.func1
        	            	  | 	external/org_golang_x_sync/errgroup/errgroup.go:78
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1695
        	            	Wraps: (4) ***UNEXPECTED ERROR; Received an unexpected execution error.
        	            	Wraps: (5) ERROR: column "col222_w0_226" is referenced by the primary key (SQLSTATE 42P10)
        	            	Error types: (1) *markers.withMark (2) *schemachange.ErrorState (3) *withstack.withStack (4) *errutil.withPrefix (5) *pgconn.PgError
        	Test:       	TestWorkload
Schema Workload Stats
Total Schema Statements Executed = 137
Total Schema Statements Succeeded = 137
Total Schema Statement Expected Failures = 0
Total Transactions Committed = 83
Total Transactions Rolled Back = 71
Total Transactions Executed = 154
    schema_change_external_test.go:96: backup, tracing data, and system table dumps in /var/lib/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/testlogs/pkg/ccl/testccl/workload/schemachange/schemachange_test/run_16_of_25/test.outputs/logTestWorkload682851827
    panic.go:626: -- test log scope end --
test logs left over in: outputs.zip/logTestWorkload682851827
--- FAIL: TestWorkload (35.87s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=16</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #134652 pkg/ccl/testccl/workload/schemachange/schemachange_test: TestWorkload failed [C-test-failure O-robot P-3 T-sql-foundations branch-release-24.1.7-rc]
- #134056 pkg/ccl/testccl/workload/schemachange/schemachange_test: TestWorkload failed [C-test-failure O-robot P-2 T-sql-foundations branch-release-24.3]
- #134046 pkg/ccl/testccl/workload/schemachange/schemachange_test: TestWorkload failed [C-test-failure O-robot P-2 T-sql-foundations branch-release-23.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestWorkload.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/ccl/testccl/workload/schemachange/schemachange_test.TestWorkload [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/17885310?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/17885310?buildTab=artifacts#/) on master @ [269d2386b39eecb9bce70c390a188a60a3265ce4](https://github.com/cockroachdb/cockroach/commits/269d2386b39eecb9bce70c390a188a60a3265ce4):


```
    "query": "SELECT $1:::REGCLASS = $2:::REGCLASS",
    "queryArgs": [
     "public.table_w1_1",
     "public.table_w1_1"
    ],
    "result": true
   }
  ],
  "previousStatements": [
   "ALTER TABLE public.table_w1_1 ADD CONSTRAINT table_w1_1_col1_w1_3_table_w1_1_col1_w1_3_fk FOREIGN KEY (col1_w1_3) REFERENCES public.table_w1_1 (col1_w1_3) ON DELETE CASCADE ON UPDATE CASCADE"
  ],
  "usesDeclarativeSchemaChanger": true
 }
}
    schema_change_external_test.go:131: 
        	Error Trace:	pkg/ccl/testccl/workload/schemachange/schema_change_external_test.go:131
        	Error:      	Received unexpected error:
        	            	***FAIL; Failed to receive an execution error when errors were expected
        	            	(1) forced error mark
        	            	  | "fatal error when running txn"
        	            	  | github.com/cockroachdb/errors/withstack/*withstack.withStack::
        	            	Wraps: (2)
        	            	Wraps: (3) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*opStmt).executeStmt
        	            	  | 	pkg/workload/schemachange/operation_generator.go:3242
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*schemaChangeWorker).runInTxn
        	            	  | 	pkg/workload/schemachange/schemachange.go:503
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*schemaChangeWorker).run
        	            	  | 	pkg/workload/schemachange/schemachange.go:600
        	            	  | pkg/ccl/testccl/workload/schemachange/schemachange_test_test.TestWorkload.TestWorkload.func4.func6
        	            	  | 	pkg/ccl/testccl/workload/schemachange/schema_change_external_test.go:120
        	            	  | golang.org/x/sync/errgroup.(*Group).Go.func1
        	            	  | 	external/org_golang_x_sync/errgroup/errgroup.go:78
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_arm64.s:1222
        	            	Wraps: (4) ***FAIL; Failed to receive an execution error when errors were expected
        	            	Error types: (1) *markers.withMark (2) *schemachange.ErrorState (3) *withstack.withStack (4) *errutil.leafError
        	Test:       	TestWorkload
Schema Workload Stats
Total Schema Statements Executed = 24
Total Schema Statements Succeeded = 24
Total Schema Statement Expected Failures = 0
Total Transactions Committed = 17
Total Transactions Rolled Back = 26
Total Transactions Executed = 43
    schema_change_external_test.go:96: backup, tracing data, and system table dumps in /artifacts/tmp/_tmp/67537770b84c3a4ea177539c1e46bd43/logTestWorkload1405471860
    panic.go:626: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/67537770b84c3a4ea177539c1e46bd43/logTestWorkload1405471860
--- FAIL: TestWorkload (10.39s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #135891 pkg/ccl/testccl/workload/schemachange/schemachange_test: TestWorkload failed [C-test-failure O-robot T-sql-foundations branch-release-24.3.0-rc release-blocker]
- #135883 pkg/ccl/testccl/workload/schemachange/schemachange_test: TestWorkload failed [C-test-failure O-robot T-sql-foundations branch-release-23.2 release-blocker]
- #134652 pkg/ccl/testccl/workload/schemachange/schemachange_test: TestWorkload failed [C-test-failure O-robot P-3 T-sql-foundations]
- #134056 pkg/ccl/testccl/workload/schemachange/schemachange_test: TestWorkload failed [column ... is referenced by the primary key] [C-test-failure O-robot P-2 T-sql-foundations branch-release-24.3]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestWorkload.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/ccl/testccl/workload/schemachange/schemachange_test.TestWorkload [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/17886939?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/17886939?buildTab=artifacts#/) on master @ [2a92c1155ebd143567fd8b2ae7bb330e3564fde7](https://github.com/cockroachdb/cockroach/commits/2a92c1155ebd143567fd8b2ae7bb330e3564fde7):


```
    "result": false
   },
   {
    "query": "SELECT ARRAY[parent.table_schema, parent.table_name, parent.column_name, child.column_name] FROM (SELECT conname, conkey, confkey, conrelid, confrelid FROM pg_constraint WHERE (contype = 'f') AND (conrelid = 'public.table_w1_1'::REGCLASS::INT8)) AS con JOIN (SELECT column_name, ordinal_position, column_default FROM information_schema.columns WHERE (table_schema = 'public') AND (table_name = 'table_w1_1')) AS child ON conkey[1] = child.ordinal_position JOIN (SELECT pc.oid, cols.table_schema, cols.table_name, cols.column_name, cols.ordinal_position FROM pg_class AS pc JOIN pg_namespace AS pn ON pc.relnamespace = pn.oid JOIN information_schema.columns AS cols ON ((pc.relname = cols.table_name) AND (pn.nspname = cols.table_schema))) AS parent ON ((con.confkey[1] = parent.ordinal_position) AND (con.confrelid = parent.oid)) WHERE child.column_name != 'rowid'",
    "result": null
   }
  ],
  "previousStatements": [
   "INSERT INTO public.table_w1_1 (col1_w1_2, col1_w1_3, col1_w1_4, col1_w1_5, col1_w1_6, col1_w1_7, col1_w1_8, col1_w1_9, col1_w1_10, col1_w1_11, col1_w1_12, col1_w1_13, col1_w1_14, col1_w1_15) VALUES ('-46 years -11 mons -914 days -13:49:15.450233':::INTERVAL,'7FFFFFFF/FFFFFFFF':::PG_LSN,701:::OID,'{\"9O\": {\"6X5\": 0.5753766976785973, \"GqixCn\": {\"4UP4Ac\": true}, \"baz\": [{\"IgP\": true, \"baz\": \"kyLS\"}, {\"EuUD\": \"4nY\"}, \"x1OqnA\", true], \"c\": \"mgo\", \"foobar\": 0.08384885004596496}, \"Zue2C\": 0.4527352670291963, \"bar\": false}':::JSONB,'4E2AD303/58A0F55A':::PG_LSN,(-17):::INT8,(-0.15404095653453065):::FLOAT8,e'\\''::BPCHAR,NULL,'0104000060E61000000600000001010000403897E6908F175C40177B8E50ABB21240D899CED7151ADD41010100004067D57B9CB332654033AA48691E4C4140CC3AD155F921F141010100004000A162F043B062C0E8AFC054BDE34FC0614641AB7027F44101010000403854A55569B85A404459ADCE615402C0979F156C7E30FB4101010000406C8CD08187733FC0B0B53C85D4A70F40D51F6DB6A6B2EAC1010100004091B8EDE0B69863404FE6D092C6FF4E40A1AE334FA2D9FCC1':::GEOGRAPHY,'0101000020E6100000511401E91CAC65403AE6C74136A04640':::GEOGRAPHY,25:::INT8,NULL,B'10011000011100101'),('-60 years -9 mons -302 days -02:17:40.556875':::INTERVAL,'2E73C8A8/5DF48093':::PG_LSN,2206:::OID,'{\"a\": null, \"foo\": [true, true], \"foobar\": \"b\", \"gf\": [[\"nn\", null], null, false, null]}':::JSONB,'812A80D4/5AB546EF':::PG_LSN,(-73):::INT8,(-0.8820704122583989):::FLOAT8,'.'::BPCHAR,'82 years 8 mons 181 days 23:44:06.623184':::INTERVAL,'01030000E0E61000000100000008000000AAFB17696B6265C0AD4575E25EA20D405DADECC45EB9FC41791B0DE06300FC41E8D451D1C38758C08581F306ADFB35C0E8B4A96ED8BCF5412638354BCAD4F3411D27C40450AD45C0188EEEF5EC6547C0530933B34E76F1414E9D9A409A20EC41624CD6EAB0960BC0E4CA7CC6416953C0F03D5B8271640042B748ED1408BCEDC1FD5E3242000C64404CFBE53B7EB73340EE3A3993A53F02420BFCB3CC5A96F2C1934CE3082582484005C2F8EC144D5040904D3A4C2F71FB419790A5C01084F5418CE666C692D34FC0BB0453D397AB55406E87FDB3381D01C273AF387E08FAF641AAFB17696B6265C0AD4575E25EA20D405DADECC45EB9FC41791B0DE06300FC41':::GEOGRAPHY,'01060000E0E61000000300000001030000C00100000009000000E65E6AF8B6B740C05D18438B18930FC01687AE058205FAC141AD1083BFFC01C2DD714F8C3FA34FC0F6E792C92B3545C02302EDFF668879410EB97C199A97D3C18537E44A63F723C09A52571A95A854C0DBA02959B32DFDC1A92AD44E7912F2C154BAFC6B90314D40AE590BB5233A51C0C1C6ABB7B101E4418FE5223A7B32D2C1DD1E87111A4A4F40C403489D2BE93DC024FF427BDD9001C2F88F8DE98A22F3C128DB9252B12A6540FE38BAC35FE44E40F17F42084394F941E6838A0BD32DF941AD9FFD58CDED614081863A9B09D54D403149DF71C00FF6C110069EC3FD4BF941B5605A1D20BC63C09F3B8A3C418F554080D9C5B5F048EA411F8294779340F541E65E6AF8B6B740C05D18438B18930FC01687AE058205FAC141AD1083BFFC01C201030000C001000000050000003EE7CE47A70D5640BF2BE276F56642C02DACC13339F0EDC1767FDD50EEC0F9C1363C7D21E08A43C06222E8DBD1C13F40599B192BE922FF41F1227356BC5DC1C1FC837C08D4AF52C0F3822623516041408070FDB7789FE9C19F745E4F84BC014208994768EA9063C0A36D937D6C1D3840879A23059028F94167B0AC3E3FA4E4C13EE7CE47A70D5640BF2BE276F56642C02DACC13339F0EDC1767FDD50EEC0F9C101030000C0010000000D000000FFF323D9822E65C061EC5671FB9A2DC085C3A5BC4714C04111CA48B4565BD4C1B7ECB4CDA15763C02625B0665A4656C02AD8C9C34D5FFE41B9E47CD4CA87F3C19DD0F4CC96D944C0FE2EFD6CB48D4CC0F943C0954FD6FA41EBB2528CE87FDC41C219ED1D5DF32B40E2A20B86754051C0B0CC062DD5CDFD41D646E1B40805F84125280849393051407DCA449FE7EB55C0BCF9968AE81DF641E12E3452D715C1419E3AB8230B525040EEDC64DE46FC30C05B53CF037681FAC1B6452A97DA0600C22BF96330DE1D5140EA8D7BA5DBD050403F70691E77ACFEC1FD5DDA1DA31E01C26954A79F014A544094F778FA9D365540E2BC9C4C8043FEC1A61EBA69E977DA41D349B06CCE335140F0AD8F524E6B5440359F42DFA8C0F8C1B694692971BFA7C1BBF46F2D53A760C0E4EBE91B31904240157041C41070FDC11E5D1CF2E813E141903217D6B34B64C0645AA7529B263E4030E95FBF4355C2C11ED1276D0032F0C100923D5A443762C0685652885225EC3F2636ADF4F2F2E3C1F314B804B453D2C1FFF323D9822E65C061EC5671FB9A2DC085C3A5BC4714C04111CA48B4565BD4C1':::GEOGRAPHY,23:::INT8,e'\\x19'::\"char\",B'11111000000011101'),('42 years 8 mons 407 days 19:58:35.233915':::INTERVAL,'7FFFFFFF/FFFFFFFF':::PG_LSN,25:::OID,'[[\"nBAz\", false], {\"43xh\": null, \"6oiAoAV9o\": [{\"WXC\": \"baz\"}, false, null, 0.14229105621372035, true], \"b\": \"baz\", \"vUV\": {\"foobar\": 0.32661456874334116, \"pYI\": false}}, {\"foobar\": null}, {\"b\": \"a\", \"foo\": {\"bar\": null, \"foobar\": \"K0D\", \"lMha0Wl\": 0.20497940534248577}}, {\"2JZsfOh\": true, \"foobar\": null}, null, null]':::JSONB,'192B4793/2EE14737':::PG_LSN,(-51):::INT8,(-0.05652063687982678):::FLOAT8,'\"'::BPCHAR,'1 day':::INTERVAL,'01010000E0E61000008528C4B01CD25F40229502EDD689464063EFE4F9174F0242936730A912ADC741':::GEOGRAPHY,'0107000020E6100000020000000101000020E6100000811B6739D33A51C058E6B0E59A8F52C00105000020E6100000010000000102000000020000001FB8D94F3DE551C0AC7DB203CF764540920D54B6564E53C086C0A99BF5183840':::GEOGRAPHY,32:::INT8,e'\\x13'::\"char\",B'11001010101000101')"
  ],
  "usesDeclarativeSchemaChanger": true
 }
}
    schema_change_external_test.go:131: 
        	Error Trace:	pkg/ccl/testccl/workload/schemachange/schema_change_external_test.go:131
        	Error:      	Received unexpected error:
        	            	***UNEXPECTED ERROR; Received an unexpected execution error.: ERROR: duplicate key value violates unique constraint "table_w1_1_col1_w1_4_unique" (SQLSTATE 23505)
        	            	(1) forced error mark
        	            	  | "fatal error when running txn"
        	            	  | github.com/cockroachdb/errors/withstack/*withstack.withStack::
        	            	Wraps: (2)
        	            	Wraps: (3) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*opStmt).executeStmt
        	            	  | 	pkg/workload/schemachange/operation_generator.go:3228
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*schemaChangeWorker).runInTxn
        	            	  | 	pkg/workload/schemachange/schemachange.go:503
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*schemaChangeWorker).run
        	            	  | 	pkg/workload/schemachange/schemachange.go:600
        	            	  | pkg/ccl/testccl/workload/schemachange/schemachange_test_test.TestWorkload.TestWorkload.func4.func6
        	            	  | 	pkg/ccl/testccl/workload/schemachange/schema_change_external_test.go:120
        	            	  | golang.org/x/sync/errgroup.(*Group).Go.func1
        	            	  | 	external/org_golang_x_sync/errgroup/errgroup.go:78
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_arm64.s:1222
        	            	Wraps: (4) ***UNEXPECTED ERROR; Received an unexpected execution error.
        	            	Wraps: (5) ERROR: duplicate key value violates unique constraint "table_w1_1_col1_w1_4_unique" (SQLSTATE 23505)
        	            	Error types: (1) *markers.withMark (2) *schemachange.ErrorState (3) *withstack.withStack (4) *errutil.withPrefix (5) *pgconn.PgError
        	Test:       	TestWorkload
Schema Workload Stats
Total Schema Statements Executed = 9
Total Schema Statements Succeeded = 9
Total Schema Statement Expected Failures = 0
Total Transactions Committed = 6
Total Transactions Rolled Back = 10
Total Transactions Executed = 16
    schema_change_external_test.go:96: backup, tracing data, and system table dumps in /artifacts/tmp/_tmp/67537770b84c3a4ea177539c1e46bd43/logTestWorkload852454429
    panic.go:626: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/67537770b84c3a4ea177539c1e46bd43/logTestWorkload852454429
--- FAIL: TestWorkload (4.55s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #135891 pkg/ccl/testccl/workload/schemachange/schemachange_test: TestWorkload failed [C-test-failure O-robot T-sql-foundations branch-release-24.3.0-rc release-blocker]
- #135883 pkg/ccl/testccl/workload/schemachange/schemachange_test: TestWorkload failed [C-test-failure O-robot T-sql-foundations branch-release-23.2 release-blocker]
- #134652 pkg/ccl/testccl/workload/schemachange/schemachange_test: TestWorkload failed [C-test-failure O-robot P-3 T-sql-foundations]
- #134056 pkg/ccl/testccl/workload/schemachange/schemachange_test: TestWorkload failed [column ... is referenced by the primary key] [C-test-failure O-robot P-2 T-sql-foundations branch-release-24.3]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestWorkload.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/ccl/testccl/workload/schemachange/schemachange_test.TestWorkload [failed](https://tanzanite.cluster.engflow.com/invocations/default/c0dc44e2-336d-49fc-b523-6d7a17138789?testReportRun=23&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL3Rlc3RjY2wvd29ya2xvYWQvc2NoZW1hY2hhbmdlOnNjaGVtYWNoYW5nZV90ZXN0) on master @ [bcc993d796d03664604bf695e38fd5644d0bc952](https://github.com/cockroachdb/cockroach/commits/bcc993d796d03664604bf695e38fd5644d0bc952):


```
    "query": "WITH tab_json AS (SELECT crdb_internal.pb_to_json('desc', descriptor)-\u003e'table' AS t FROM system.descriptor WHERE id = $1::REGCLASS), columns_json AS (SELECT json_array_elements(t-\u003e'columns') AS c FROM tab_json), columns AS (SELECT c-\u003e\u003e'computeExpr' AS generation_expression, c-\u003e\u003e'name' AS column_name, c-\u003e\u003e'id' AS ordinal FROM columns_json) SELECT generation_expression FROM columns WHERE generation_expression IS NOT NULL",
    "queryArgs": [
     "schema_w0_85.table_w0_94"
    ],
    "result": []
   }
  ],
  "previousStatements": [
   "ALTER TABLE schema_w0_85.table_w0_94 DROP COLUMN col94_w0_96"
  ],
  "usesDeclarativeSchemaChanger": true
 }
}
    schema_change_external_test.go:131: 
        	Error Trace:	pkg/ccl/testccl/workload/schemachange/schema_change_external_test.go:131
        	Error:      	Received unexpected error:
        	            	***UNEXPECTED ERROR; Received an unexpected execution error.: ERROR: column "col94_w0_96" is referenced by the primary key (SQLSTATE 42P10)
        	            	(1) forced error mark
        	            	  | "fatal error when running txn"
        	            	  | github.com/cockroachdb/errors/withstack/*withstack.withStack::
        	            	Wraps: (2)
        	            	Wraps: (3) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*opStmt).executeStmt
        	            	  | 	pkg/workload/schemachange/operation_generator.go:3231
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*schemaChangeWorker).runInTxn
        	            	  | 	pkg/workload/schemachange/schemachange.go:503
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*schemaChangeWorker).run
        	            	  | 	pkg/workload/schemachange/schemachange.go:600
        	            	  | pkg/ccl/testccl/workload/schemachange/schemachange_test_test.TestWorkload.TestWorkload.func4.func6
        	            	  | 	pkg/ccl/testccl/workload/schemachange/schema_change_external_test.go:120
        	            	  | golang.org/x/sync/errgroup.(*Group).Go.func1
        	            	  | 	external/org_golang_x_sync/errgroup/errgroup.go:78
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1695
        	            	Wraps: (4) ***UNEXPECTED ERROR; Received an unexpected execution error.
        	            	Wraps: (5) ERROR: column "col94_w0_96" is referenced by the primary key (SQLSTATE 42P10)
        	            	Error types: (1) *markers.withMark (2) *schemachange.ErrorState (3) *withstack.withStack (4) *errutil.withPrefix (5) *pgconn.PgError
        	Test:       	TestWorkload
Schema Workload Stats
Total Schema Statements Executed = 103
Total Schema Statements Succeeded = 103
Total Schema Statement Expected Failures = 0
Total Transactions Committed = 59
Total Transactions Rolled Back = 70
Total Transactions Executed = 129
    schema_change_external_test.go:96: backup, tracing data, and system table dumps in /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/testlogs/pkg/ccl/testccl/workload/schemachange/schemachange_test/run_23_of_25/test.outputs/logTestWorkload2532531546
    panic.go:626: -- test log scope end --
test logs left over in: outputs.zip/logTestWorkload2532531546
--- FAIL: TestWorkload (12.73s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=23</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #136308 pkg/ccl/testccl/workload/schemachange/schemachange_test: TestWorkload failed [C-test-failure O-robot T-sql-foundations branch-release-24.3.0-rc release-blocker]
- #136305 pkg/ccl/testccl/workload/schemachange/schemachange_test: TestWorkload failed [C-test-failure O-robot T-sql-foundations branch-release-24.3.1-rc release-blocker]
- #134056 pkg/ccl/testccl/workload/schemachange/schemachange_test: TestWorkload failed [column ... is referenced by the primary key] [C-test-failure O-robot P-2 T-sql-foundations branch-release-24.3]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestWorkload.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/workload/schemachange/error_screening.go
+++ b/pkg/workload/schemachange/error_screening.go
@@ -1417,35 +1417,42 @@ func (og *operationGenerator) tableHasOngoingAlterPKSchemaChanges(
 		ctx,
 		tx,
 		`
-WITH
-	descriptors
-		AS (
-			SELECT
-				crdb_internal.pb_to_json(
-					'cockroach.sql.sqlbase.Descriptor',
-					descriptor
-				)->'table'
-					AS d
-			FROM
-				system.descriptor
-			WHERE
-				id = $1::REGCLASS
-		)
-SELECT
-	EXISTS(
-		SELECT
-			mut
-		FROM
-			(
-				SELECT
-					json_array_elements(d->'mutations')
-						AS mut
-				FROM
-					descriptors
-			)
-		WHERE
-			(mut->'primaryKeySwap') IS NOT NULL
-	);
+WITH descriptors AS (
+                    SELECT crdb_internal.pb_to_json(
+                            'cockroach.sql.sqlbase.Descriptor',
+                            descriptor
+                           )->'table' AS d
+                      FROM system.descriptor
+                     WHERE id = $1::REGCLASS
+                 ),
+     mutations AS (
+                SELECT json_array_elements(
+                        d->'mutations'
+                       ) AS m
+                  FROM descriptors
+               ),
+     primaryindex AS (
+                    SELECT d->'primaryIndex' AS p
+                      FROM descriptors
+                  )
+		 -- Check for legacy primary key swaps which exist as mutations
+     SELECT EXISTS(
+                SELECT mut
+                  FROM (
+                        SELECT json_array_elements(
+                                d->'mutations'
+                               ) AS mut
+                          FROM descriptors
+                       )
+                 WHERE (mut->'primaryKeySwap') IS NOT NULL
+            )
+		 -- Check for declarative primary key swaps, which will appear as
+		 -- as new primary indexes with different key columns
+     UNION SELECT count(*) > 0
+             FROM primaryindex AS pk, mutations AS mut
+            WHERE m->'index'->'encodingType' = '1'::JSONB
+                  AND m->'index'->'keyColumnIds'
+                    != p->'keyColumnIds';
 		`,
 		tableName.String(),
 	)

--- a/pkg/workload/schemachange/operation_generator.go
+++ b/pkg/workload/schemachange/operation_generator.go
@@ -306,7 +306,7 @@ func (og *operationGenerator) addColumn(ctx context.Context, tx pgx.Tx) (*opStmt
 		{code: pgcode.DuplicateColumn, condition: columnExistsOnTable},
 		{code: pgcode.UndefinedObject, condition: typ == nil},
 		{code: pgcode.NotNullViolation, condition: hasRows && def.Nullable.Nullability == tree.NotNull},
-		{code: pgcode.FeatureNotSupported, condition: hasAlterPKSchemaChange},
+		{code: pgcode.FeatureNotSupported, condition: hasAlterPKSchemaChange && !og.useDeclarativeSchemaChanger},
 		// UNIQUE is only supported for indexable types.
 		{
 			code:      pgcode.FeatureNotSupported,
@@ -398,7 +398,7 @@ func (og *operationGenerator) addUniqueConstraint(ctx context.Context, tx pgx.Tx
 		{code: pgcode.UndefinedColumn, condition: !columnExistsOnTable},
 		{code: pgcode.DuplicateRelation, condition: constraintExists},
 		{code: pgcode.FeatureNotSupported, condition: columnExistsOnTable && !colinfo.ColumnTypeIsIndexable(columnForConstraint.typ)},
-		{pgcode.FeatureNotSupported, hasAlterPKSchemaChange},
+		{pgcode.FeatureNotSupported, hasAlterPKSchemaChange && !og.useDeclarativeSchemaChanger},
 		{code: pgcode.ObjectNotInPrerequisiteState, condition: databaseHasRegionChange && tableIsRegionalByRow},
 	})
 
@@ -1146,7 +1146,7 @@ func (og *operationGenerator) createIndex(ctx context.Context, tx pgx.Tx) (*opSt
 			{code: pgcode.FeatureNotSupported, condition: regionColStored},
 			{code: pgcode.FeatureNotSupported, condition: duplicateRegionColumn},
 			{code: pgcode.FeatureNotSupported, condition: isStoringVirtualComputed},
-			{code: pgcode.FeatureNotSupported, condition: hasAlterPKSchemaChange},
+			{code: pgcode.FeatureNotSupported, condition: hasAlterPKSchemaChange && !og.useDeclarativeSchemaChanger},
 			{code: pgcode.FeatureNotSupported, condition: lastColInvertedIndexIsDescending},
 			{code: pgcode.FeatureNotSupported, condition: pkColUsedInInvertedIndex},
 		})
@@ -1678,13 +1678,16 @@ func (og *operationGenerator) dropColumn(ctx context.Context, tx pgx.Tx) (*opStm
 		{code: pgcode.UndefinedColumn, condition: !columnExists},
 		{code: pgcode.InvalidColumnReference, condition: colIsPrimaryKey || colIsRefByComputed},
 		{code: pgcode.DependentObjectsStillExist, condition: columnIsDependedOn || columnRemovalWillDropFKBackingIndexes},
-		{code: pgcode.FeatureNotSupported, condition: hasAlterPKSchemaChange},
+		{code: pgcode.FeatureNotSupported, condition: hasAlterPKSchemaChange && !og.useDeclarativeSchemaChanger},
 	})
-	// For legacy schema changer its possible for create index operations to interfere if they
-	// are in progress.
-	if !og.useDeclarativeSchemaChanger {
-		stmt.potentialExecErrors.add(pgcode.ObjectNotInPrerequisiteState)
-	}
+	stmt.potentialExecErrors.addAll(codesWithConditions{
+		// For legacy schema changer its possible for create index operations to interfere if they
+		// are in progress.
+		{code: pgcode.ObjectNotInPrerequisiteState, condition: !og.useDeclarativeSchemaChanger},
+		// It is possible the column we are dropping is in the new primary key,
+		// so a potential error is an invalid reference in this case.
+		{code: pgcode.InvalidColumnReference, condition: og.useDeclarativeSchemaChanger && hasAlterPKSchemaChange},
+	})
 	stmt.sql = fmt.Sprintf(`ALTER TABLE %s DROP COLUMN %s`, tableName.String(), columnName.String())
 	return stmt, nil
 }
@@ -1922,7 +1925,7 @@ func (og *operationGenerator) dropIndex(ctx context.Context, tx pgx.Tx) (*opStmt
 	if err != nil {
 		return nil, err
 	}
-	if hasAlterPKSchemaChange {
+	if hasAlterPKSchemaChange && !og.useDeclarativeSchemaChanger {
 		stmt.expectedExecErrors.add(pgcode.FeatureNotSupported)
 	}
 

